Documentation Simon Pfaff Master Bioinformatic
Start 05.02.18




* DONE [#A] Week 1 05.02.18 - 09.02.18
  - State "DONE"       from "PROGRESS"   [2018-02-14 Mi 14:32]
  - State "PROGRESS"   from "TODO"       [2018-02-06 Di 14:41]
** Meeting
   https://github.com/chloroExtractorTeam/chloroplast_landscape/issues/7

** DONE Build Dockercontainer
   - State "DONE"       from "PROGRESS"   [2018-02-08 Do 15:10]
   - State "PROGRESS"   from "TODO"       [2018-02-06 Di 14:38]
*** DONE chloroExtractor
    - State "DONE"       from "PROGRESS"   [2018-02-06 Di 14:37]
    - State "PROGRESS"   from "TODO"       [2018-02-06 Di 14:37]
      
      Prebuild by markus
   
*** DONE Org.asm
    - State "DONE"       from "PROGRESS"   [2018-02-06 Di 14:37]
    - State "PROGRESS"   from "TODO"       [2018-02-06 Di 14:37]

      Prebuilt by markus

*** DONE NOVOPlasty
    - State "DONE"       from "PROGRESS"   [2018-02-06 Di 14:37]
    - State "PROGRESS"   from "TODO"       [2018-02-06 Di 14:37]

      built together on Monday, 05.02

*** DONE fast-plast
    - State "DONE"       from "PROGRESS"   [2018-02-06 Di 14:37]
    - State "PROGRESS"   from "TODO"       [2018-02-06 Di 14:37]

      prebuilt by markus

*** DONE getOrganelle
    - State "DONE"       from "PROGRESS"   [2018-02-07 Mi 14:16]
    - State "PROGRESS"   from "TODO"       [2018-02-06 Di 14:37]

    Dependencies: SPAdes, bowtie2, BLAST

*** DONE IOGA 
    - State "DONE"       from "PROGRESS"   [2018-02-08 Do 15:07]
    - State "PROGRESS"   from "TODO"       [2018-02-06 Di 16:57]
    
    Dependencies: Python, BioPython, BBmap, SOAPdenovo2, SeqTK, SPAdes.py, ALE, Samtools 0.1.19, Picardtools
    has an install script for all dep.

    i hate python... -.-

    fu* python, who changes syntax -.-

    Problem: script can't import wget..
     -> pip intstall -U wget
     still can not import wget

     python3 can import wget, but don't run the script 'cuz it hast python2 syntax of print .....

     Final solution provided by Frank


** CANCELED [#C] All Dockers in one Repo
   - State "CANCELED"   from ""           [2018-02-12 Mo 10:19]
     
     Issue moved to "Less importent stuff"
      

** DONE [#A] build run scripts and wrapper for all programs
   - State "DONE"       from "PROGRESS"   [2018-02-14 Mi 14:32]
   - State "PROGRESS"   from "TODO"       [2018-02-08 Do 15:07]

*** DONE Run for GetOrganelle
    - State "DONE"       from "PROGRESS"   [2018-02-08 Do 15:40]
    - State "PROGRESS"   from "TODO"       [2018-02-08 Do 15:07]

**** DONE Test run on Testdata
     - State "DONE"       from "PROGRESS"   [2018-02-08 Do 15:40]
     - State "PROGRESS"   from "TODO"       [2018-02-08 Do 15:24]

*** DONE Run for IOGA
    - State "DONE"       from "PROGRESS"   [2018-02-08 Do 16:22]
    - State "PROGRESS"   from "TODO"       [2018-02-08 Do 15:40]
**** DONE Test run for IOGA
     - State "DONE"       from "PROGRESS"   [2018-02-08 Do 16:22]
     - State "PROGRESS"   from "TODO"       [2018-02-08 Do 15:58]

*** DONE Run for NOVOPlasty
    - State "DONE"       from "PROGRESS"   [2018-02-14 Mi 14:31]
    - State "PROGRESS"   from "TODO"       [2018-02-14 Mi 14:31]
    needs his file paths in a config file....

*** CANCELED Write Wrapper
    - State "CANCELED"   from ""           [2018-02-14 Mi 14:32]
      look at week 3


** DONE running SRAs over weekend 10.2 - 12.2 
   - State "DONE"       from "PROGRESS"   [2018-02-12 Mo 10:10]
   - State "PROGRESS"   from "TODO"       [2018-02-09 Fr 13:45]

     Dir: /home/s229502/Master_thesis/FirstTest_9-2

     startet three SSRs for GO, CE, IOGA, O.G, FP
     
     none of them finished...


* DONE Week 2 12.02.18 - 16.02.18
  - State "DONE"       from "PROGRESS"   [2018-02-20 Di 10:13]
  - State "PROGRESS"   from "TODO"       [2018-02-12 Mo 10:12]

** Meeting
   https://github.com/chloroExtractorTeam/chloroplast_landscape/issues/8

** DONE [#A] Evaluate the test runs, why none finished?
   - State "DONE"       from "PROGRESS"   [2018-02-20 Di 10:13]
   - State "PROGRESS"   from "TODO"       [2018-02-12 Mo 10:13]

     dir: /home/s229502/Master_thesis/FirstTest_9-2

*** chloroExtractor
**** Testset SRR5216995 / slurm-31034.out *WORKED*
-> Docker runs with testset SRR5216995, finished!

**** SRR1945435 / slurm-30871.out *ERROR*
    Reading seed kmers
    Reading total kmers
    Plotting seed and total kmers
    Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
    Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
    Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
    [18-02-09 15:12:46] [PipeWrap] scr exited:65280 
    [18-02-09 15:12:46] [PipeWrap] HASH(0x20d09a8) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.     
**** SRR1945436 / slurm-30876.out *ERROR*
     
     Reading seed kmers
     Reading total kmers
     Plotting seed and total kmers
     Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
     Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
     Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
     [18-02-09 15:34:31] [PipeWrap] scr exited:65280 
     [18-02-09 15:34:31] [PipeWrap] HASH(0x35eb9d8) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.

**** SRR1945437 / slurm-30877.out *ERROR*

     Reading seed kmers
     Reading total kmers
     Plotting seed and total kmers
     Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
     Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
     Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
     [18-02-09 15:38:04] [PipeWrap] scr exited:65280 
     [18-02-09 15:38:04] [PipeWrap] HASH(0x2a719d8) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.


*** IOGA

    All give the error that they don't find a dir, "/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR*..." - but the program writes to  "/home/s229502/Master_thesis/FirstTest_9-2/data_9-2SRR*", a bug?

**** Testset  SRR5216995 / slurm-31044.out *worked*
     Same Error, maybe a bug....
     
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR5216995_1M_1.fastq_IOGA_OUT.final/SRR5216995_1M_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR5216995_1M_1.fastq_IOGA_OUT.soap.ctg  -2502790.673759 3 128507 1 84317 has the highest ALE score
     [IOGA] Started  16:42:37.176055
     [IOGA] Finished 17:35:27.600898


**** SRR1945435 / slurm-30874.out *ERROR*

     Traceback (most recent call last):
  File "/IOGA/IOGA.py", line 432, in <module>
    main(args.reference, args.name, args.forward, args.reverse, args.threads, args.insertsize, args.maxrounds, args.verbose)
  File "/IOGA/IOGA.py", line 377, in main
    source,FP,RP,final_iteration = IOGA_loop(name,ref,forward,reverse,insertsize,threads,maxrounds)
  File "/IOGA/IOGA.py", line 322, in IOGA_loop
    shutil.rmtree(name + '.' + str(iteration-2))
  File "/usr/lib/python2.7/shutil.py", line 239, in rmtree
    onerror(os.listdir, path, sys.exc_info())
  File "/usr/lib/python2.7/shutil.py", line 237, in rmtree
    names = os.listdir(path)
OSError: [Errno 2] No such file or directory: '/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR1945435_1.fastq_IOGA_OUT.1'

**** SRR1945436 / slurm-30882.out *ERROR*

     Traceback (most recent call last):
  File "/IOGA/IOGA.py", line 432, in <module>
    main(args.reference, args.name, args.forward, args.reverse, args.threads, args.insertsize, args.maxrounds, args.verbose)
  File "/IOGA/IOGA.py", line 377, in main
    source,FP,RP,final_iteration = IOGA_loop(name,ref,forward,reverse,insertsize,threads,maxrounds)
  File "/IOGA/IOGA.py", line 322, in IOGA_loop
    shutil.rmtree(name + '.' + str(iteration-2))
  File "/usr/lib/python2.7/shutil.py", line 239, in rmtree
    onerror(os.listdir, path, sys.exc_info())
  File "/usr/lib/python2.7/shutil.py", line 237, in rmtree
    names = os.listdir(path)
OSError: [Errno 2] No such file or directory: '/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR1945436_1.fastq_IOGA_OUT.1'

**** SRR1945437 / slurm-30883.out *ERROR*
     
     Traceback (most recent call last):
  File "/IOGA/IOGA.py", line 432, in <module>
    main(args.reference, args.name, args.forward, args.reverse, args.threads, args.insertsize, args.maxrounds, args.verbose)
  File "/IOGA/IOGA.py", line 377, in main
    source,FP,RP,final_iteration = IOGA_loop(name,ref,forward,reverse,insertsize,threads,maxrounds)
  File "/IOGA/IOGA.py", line 322, in IOGA_loop
    shutil.rmtree(name + '.' + str(iteration-2))
  File "/usr/lib/python2.7/shutil.py", line 239, in rmtree
    onerror(os.listdir, path, sys.exc_info())
  File "/usr/lib/python2.7/shutil.py", line 237, in rmtree
    names = os.listdir(path)
OSError: [Errno 2] No such file or directory: '/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR1945437_1.fastq_IOGA_OUT.1'




*** Fast-plast
**** DONE Docker problem?
     - State "DONE"       from "PROGRESS"   [2018-02-16 Fr 12:59]
     - State "PROGRESS"   from "TODO"       [2018-02-12 Mo 13:02]
  #  None Information in slurm files, no other log data found, only trimmed reads and a mapping file.
  #  maybe finished?
  #  no log at all?
  #
  #  --> Problem maybe with Docker / Singularity writeable rights
  #  Markus is looking into it
**** SRRSRR5216995
    Fri Feb 16 11:03:17 2018        Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
                                    Coverage analysis was not selected.  We highly recommend this option to verify the assembly.
				    
*** GetOrganelle

**** Testset SRR5216995 / slurm-31036.out *WORKED*
     Finisehd, but you have to assamble the contigs of the ring by yourself.... 
     -> maybe Frank's find_cyclic skirpt?

**** SRR1945435 / slurm-30873.out *ERROR*
     
     2018-02-09 15:16:11,572 - INFO: Assembling with SPAdes ...
     2018-02-09 15:16:13,056 - ERROR: Error in SPAdes: 
     == Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR1945435_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


     2018-02-09 15:16:13,057 - ERROR: Assembling failed.
     
**** SRR1945436 / slurm-30878.out *ERROR*
     2018-02-09 15:41:06,317 - INFO: Assembling with SPAdes ...
     2018-02-09 15:41:08,166 - ERROR: Error in SPAdes: 
     == Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR1945436_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


     2018-02-09 15:41:08,168 - ERROR: Assembling failed.

**** SRR1945437 / slurm-30879.out *ERROR*
     
     2018-02-09 15:44:09,739 - INFO: Assembling with SPAdes ...
     2018-02-09 15:44:11,767 - ERROR: Error in SPAdes: 
     == Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR1945437_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255
     

     2018-02-09 15:44:11,769 - ERROR: Assembling failed.
     

*** Org.asm
**** DONE Docker problem
     - State "DONE"       from "PROGRESS"   [2018-02-12 Mo 13:11]
     - State "PROGRESS"   from "TODO"       [2018-02-12 Mo 14:00]
    
     Docker does not work, maybe PATH not set correctly by Org.asm after installation?

      Creating container runtime...
      /.singularity.d/actions/exec: 9: exec: oa: not found
      Docker image path: index.docker.io/chloroextractorteam/org.asm_docker:latest
      Cache folder set to /home/s229502/.singularity/docker
      Creating container runtime...
      /.singularity.d/actions/exec: 9: exec: oa: not found
      /.singularity.d/actions/exec: 9: exec: oa: not found

      -> Markus and I fixed the Bug, it was a path problem after a version update:

**** Testdata SRR5216995 / slurm-31039.out *WORKED*
     Finished w/o problems - got the circular path

**** SRR1945435 / slurm-31048.out *ERROR*
     
Traceback (most recent call last):
  File "/usr/bin/oa", line 64, in <module>
    config[root_config_name]['module'].run(config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 149, in run
    asm = restoreGraph(output+'.oax',r,x)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 2068, in restoreGraph
    filein = open(filein)
FileNotFoundError: [Errno 2] No such file or directory: '/home/s229502/Master_thesis/FirstTest_9-2/data_9-2/SRR1945435_1.fastq_oa_test.chl.oas/assembling.oax'

**** SRR1945436 / slurm-31049.out *ERROR*
     Traceback (most recent call last):
  File "/usr/bin/oa", line 64, in <module>
    config[root_config_name]['module'].run(config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 194, in run
    paths=buildPath(asm, cg, back, minlink, config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 113, in buildPath
    back=int(back),minlink=minlink)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 1665, in pathConstraints
    constraints = mergeConstraints(constraints,minisatDoubleConstraints(self,cg))
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 1532, in minisatDoubleConstraints
    r1 = round(middleFcov/float(inputcov))
ZeroDivisionError: float division by zero

**** SRR1945437 / slurm-31050.out *WORKED*
     Finished w/o problems and got the linear path
     

*** NOVOPlasty

**** Testdata SRR5216995 / slurm-31046.out *WORKED*
     Start Assembly...


-----------------Assembly 1 finished successfully: The genome has been circularized-----------------

Contig 1                  : 154479 bp

Total contigs              : 1
Largest contig             : 154479 bp
Smallest contig            : 154479 bp
Average insert size        : 312 bp


-----------------------------------------Input data metrics-----------------------------------------

Total reads                : 1999718
Aligned reads              : 68422
Assembled reads            : 58906
Organelle genome %         : 3.42 %
Average organelle coverage : 133

----------------------------------------------------------------------------------------------------


Thank you for using NOVOPlasty!

**** SRR1945435 / slurm-31175.out *ERROR*
     Building Hash Table......OK

     Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.

**** SRR1945436 / slurm-31176.out *ERROR*
     Building Hash Table......OK

     Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.

**** SRR1945437 /  slurm-31177.out *ERROR*
     Building Hash Table......OK

     Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.



     
** DONE Start new run with test-data SRR5216995_1M.fastq
   - State "DONE"       from "PROGRESS"   [2018-02-13 Di 10:18]
   - State "PROGRESS"   from "TODO"       [2018-02-12 Mo 10:14]
     
   except IOGA (and Fast-plast (pending)) all tools got the chloroplast from the Demo set 



** DONE Maybe a data problem, mapping the 3 tested SRRs on a chloroplast to find out weather the reads are maybe sorted
   - State "DONE"       from "PROGRESS"   [2018-02-13 Di 13:09]
   - State "PROGRESS"   from "TODO"       [2018-02-12 Mo 15:20]
     
     -Make Bowtie index on TAIR10_chC
     -Map all 3 on the chloroplast

     - dir: /home/s229502/Master_thesis/mapping_chloro

*** SRR1945435 / *19.95% overall alignment rate*

    2000000 reads; of these:
    2000000 (100.00%) were paired; of these:
    1614057 (80.70%) aligned concordantly 0 times
    232033 (11.60%) aligned concordantly exactly 1 time
    153910 (7.70%) aligned concordantly >1 times
    ----
    1614057 pairs aligned concordantly 0 times; of these:
      6049 (0.37%) aligned discordantly 1 time
    ----
    1608008 pairs aligned 0 times concordantly or discordantly; of these:
      3216016 mates make up the pairs; of these:
        3202005 (99.56%) aligned 0 times
        3080 (0.10%) aligned exactly 1 time
        10931 (0.34%) aligned >1 times
    19.95% overall alignment rate

    
*** SRR1945436 / *23.85% overall alignment rate*
    2000000 reads; of these:
    2000000 (100.00%) were paired; of these:
    1539334 (76.97%) aligned concordantly 0 times
    277214 (13.86%) aligned concordantly exactly 1 time
    183452 (9.17%) aligned concordantly >1 times
    ----
    1539334 pairs aligned concordantly 0 times; of these:
      7937 (0.52%) aligned discordantly 1 time
    ----
    1531397 pairs aligned 0 times concordantly or discordantly; of these:
      3062794 mates make up the pairs; of these:
        3045823 (99.45%) aligned 0 times
        3010 (0.10%) aligned exactly 1 time
        13961 (0.46%) aligned >1 times
    23.85% overall alignment rate


    
*** SRR1945437 / *24.56% overall alignment rate*
    2000000 reads; of these:
    2000000 (100.00%) were paired; of these:
    1528641 (76.43%) aligned concordantly 0 times
    288135 (14.41%) aligned concordantly exactly 1 time
    183224 (9.16%) aligned concordantly >1 times
    ----
    1528641 pairs aligned concordantly 0 times; of these:
      9982 (0.65%) aligned discordantly 1 time
    ----
    1518659 pairs aligned 0 times concordantly or discordantly; of these:
      3037318 mates make up the pairs; of these:
        3017525 (99.35%) aligned 0 times
        3352 (0.11%) aligned exactly 1 time
        16441 (0.54%) aligned >1 times
    24.56% overall alignment rate





*** SRR5216995 / *10.60% overall alignment rate*
    1000000 reads; of these:
    1000000 (100.00%) were paired; of these:
    919781 (91.98%) aligned concordantly 0 times
    55464 (5.55%) aligned concordantly exactly 1 time
    24755 (2.48%) aligned concordantly >1 times
    ----
    919781 pairs aligned concordantly 0 times; of these:
      15731 (1.71%) aligned discordantly 1 time
    ----
    904050 pairs aligned 0 times concordantly or discordantly; of these:
      1808100 mates make up the pairs; of these:
        1788073 (98.89%) aligned 0 times
        2751 (0.15%) aligned exactly 1 time
        17276 (0.96%) aligned >1 times
10.60% overall alignment rate




** DONE Run on Sim-Reads
   - State "DONE"       from "PROGRESS"   [2018-02-19 Mo 11:31]
   - State "PROGRESS"   from "TODO"       [2018-02-13 Di 15:14]
     
*** Testmappings [Genom:Chloroplast]
**** 1:10 *1.30% overall alignment rate*
     2000000 reads; of these:
     2000000 (100.00%) were paired; of these:
     1987341 (99.37%) aligned concordantly 0 times
     8446 (0.42%) aligned concordantly exactly 1 time
     4213 (0.21%) aligned concordantly >1 times
     ----
     1987341 pairs aligned concordantly 0 times; of these:
     8670 (0.44%) aligned discordantly 1 time
     ----
     1978671 pairs aligned 0 times concordantly or discordantly; of these:
     3957342 mates make up the pairs; of these:
        3948066 (99.77%) aligned 0 times
        272 (0.01%) aligned exactly 1 time
        9004 (0.23%) aligned >1 times
     1.30% overall alignment rate

**** 1:100 *11.51% overall alignment rate*
     2000000 reads; of these:
     2000000 (100.00%) were paired; of these:
     1885616 (94.28%) aligned concordantly 0 times
     75787 (3.79%) aligned concordantly exactly 1 time
     38597 (1.93%) aligned concordantly >1 times
     ----
     1885616 pairs aligned concordantly 0 times; of these:
     75678 (4.01%) aligned discordantly 1 time
     ----
     1809938 pairs aligned 0 times concordantly or discordantly; of these:
     3619876 mates make up the pairs; of these:
        3539661 (97.78%) aligned 0 times
        1675 (0.05%) aligned exactly 1 time
        78540 (2.17%) aligned >1 times
     11.51% overall alignment rate

**** 1:1000 *54.16% overall alignment rate*
     2000000 reads; of these:
     2000000 (100.00%) were paired; of these:
     1461704 (73.09%) aligned concordantly 0 times
     356693 (17.83%) aligned concordantly exactly 1 time
     181603 (9.08%) aligned concordantly >1 times
     ----
     1461704 pairs aligned concordantly 0 times; of these:
     356108 (24.36%) aligned discordantly 1 time
     ----
     1105596 pairs aligned 0 times concordantly or discordantly; of these:
     2211192 mates make up the pairs; of these:
     1833404 (82.91%) aligned 0 times
     7471 (0.34%) aligned exactly 1 time
     370317 (16.75%) aligned >1 times
     54.16% overall alignment rate
     

*** CE
**** 1:10 *ERROR - n.e. coverage*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
[18-02-14 13:58:35] [scr] Insufficient plastid coverage detected: 39X. At least 200X required
[18-02-14 13:58:35] [scr] You might need to increase the amount of input data
[18-02-14 13:58:35] [scr] If values are very low, make sure your library contains plastid reads
[18-02-14 13:58:36] [PipeWrap] scr exited:256 
[18-02-14 13:58:36] [PipeWrap] HASH(0x38cb180) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.
**** 1:100 *worked - circular*
[18-02-14 14:26:22] [fcg] Found 3 different nodes
[18-02-14 14:26:22] [fcg] The inverted repeat is node number: 2
[18-02-14 14:26:22] [fcg] The LSC is node number: 1 and the SSC is node number: 3
[18-02-14 14:26:22] [fcg] Single circular chloroplast seems to be found
[18-02-14 14:26:22] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
[18-02-14 14:26:23] [PipeWrap] ptx pipeline completed

**** 1:1000 *worked - circular*
[18-02-14 14:38:59] [fcg] Found 3 different nodes
[18-02-14 14:38:59] [fcg] The inverted repeat is node number: 1
[18-02-14 14:38:59] [fcg] The LSC is node number: 0 and the SSC is node number: 2
[18-02-14 14:38:59] [fcg] Single circular chloroplast seems to be found
[18-02-14 14:38:59] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
[18-02-14 14:39:00] [PipeWrap] ptx pipeline completed

*** GO
**** 1:10 *ERROR*
2018-02-14 14:34:55,359 - INFO: Assembling with SPAdes ...
2018-02-14 14:34:58,208 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/test_sim/data_2M/sim_1_10_2M/sim_1_10_2M_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 14:34:58,209 - ERROR: Assembling failed.

**** 1:100 *ERROR*
2018-02-14 14:17:56,957 - INFO: Assembling with SPAdes ...
2018-02-14 14:17:59,560 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/test_sim/data_2M/sim_1_100_2M/sim_1_100_2M_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 14:17:59,561 - ERROR: Assembling failed.

**** 1:1000 *ERROR*
2018-02-14 14:23:34,194 - INFO: Assembling with SPAdes ...
2018-02-14 14:23:37,944 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/test_sim/data_2M/sim_1_1000_2M/sim_1_1000_2M_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 14:23:37,945 - ERROR: Assembling failed.
    
*** IOGA
**** 1:10 *ERROR*
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/test_sim/data_2M/sim_1_10_2M/sim_1_10_2M_1.fastq_IOGA_OUT.final/sim_1_10_2M_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     Traceback (most recent call last):
     File "/IOGA/IOGA.py", line 432, in <module>
     main(args.reference, args.name, args.forward, args.reverse, args.threads, args.insertsize, args.maxrounds, args.verbose)
     File "/IOGA/IOGA.py", line 407, in main
     print '[IOGA] {} {} {} {} {} {} has the highest ALE score'.format(*ALE_score[0])
     IndexError: list index out of range
**** 1:100     
**** 1:1000

*** O.A
**** 1:10 *worked - circular*
2018-02-14 14:24:54,047 [INFO ]  Minimum stem coverage = 37
2018-02-14 14:24:54,772 [INFO ]  Both segments -2 and -1 are connected (paired-end=46 frg length=521.282609 sd=132.905089)
2018-02-14 14:24:54,775 [INFO ]  Both segments -1 and 3 are connected (paired-end=48 frg length=519.187500 sd=100.379208)
2018-02-14 14:24:54,778 [INFO ]  Both segments 3 and 1 are connected (paired-end=50 frg length=523.120000 sd=93.818896)
2018-02-14 14:24:54,782 [INFO ]  Path is circular and connected by 62  (length: 570, sd: 126)

**** 1:100 *worked - circular*
2018-02-14 14:32:50,978 [INFO ]  Minimum stem coverage = 327
2018-02-14 14:32:53,070 [INFO ]  Both segments -3 and 1 are connected (paired-end=347 frg length=535.867435 sd=120.084483)
2018-02-14 14:32:53,084 [INFO ]  Both segments 1 and -2 are connected (paired-end=308 frg length=540.691558 sd=110.788934)
2018-02-14 14:32:53,096 [INFO ]  Both segments -2 and -1 are connected (paired-end=313 frg length=556.968051 sd=121.149088)
2018-02-14 14:32:53,109 [INFO ]  Path is circular and connected by 311  (length: 554, sd: 125)

**** 1:1000 *worked - circular*
2018-02-14 14:45:03,567 [INFO ]  Minimum stem coverage = 1562
2018-02-14 14:45:10,731 [INFO ]  Both segments -3 and 1 are connected (paired-end=1301 frg length=545.867025 sd=126.561101)
2018-02-14 14:45:10,766 [INFO ]  Both segments 1 and -2 are connected (paired-end=1247 frg length=547.178829 sd=128.751784)
2018-02-14 14:45:10,796 [INFO ]  Both segments -2 and -1 are connected (paired-end=1258 frg length=537.937202 sd=126.617321)
2018-02-14 14:45:10,831 [INFO ]  Path is circular and connected by 1229  (length: 537, sd: 127)

*** NP
**** 1:10 *worked - 3 contigs*
------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------

Contig 01                  : 136698 bp
Contig 02                  : 18656 bp
Contig 03                  : 18830 bp

Total contigs              : 3
Largest contig             : 136698 bp
Smallest contig            : 18656 bp
Average insert size        : 446 bp


-----------------------------------------Input data metrics-----------------------------------------

Total reads                : 3999984
Aligned reads              : 48242
Assembled reads            : 40442
Organelle genome %         : 1.21 %
Average organelle coverage : 94

----------------------------------------------------------------------------------------------------

**** 1:100 *worked - 3 contigs*
------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------

Contig 01                  : 136698 bp
Contig 02                  : 18773 bp
Contig 03                  : 18651 bp

Total contigs              : 3
Largest contig             : 136698 bp
Smallest contig            : 18651 bp
Average insert size        : 397 bp


-----------------------------------------Input data metrics-----------------------------------------

Total reads                : 3999982
Aligned reads              : 427364
Assembled reads            : 272742
Organelle genome %         : 10.68 %
Average organelle coverage : 830

----------------------------------------------------------------------------------------------------
**** 1:1000 *worked - 3 contigs*
     Start Assembly...


------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------

Contig 01                  : 136698 bp
Contig 02                  : 18642 bp
Contig 03                  : 18764 bp

Total contigs              : 3
Largest contig             : 136698 bp
Smallest contig            : 18642 bp
Average insert size        : 398 bp


-----------------------------------------Input data metrics-----------------------------------------

Total reads                : 3999998
Aligned reads              : 2013610
Assembled reads            : 729016
Organelle genome %         : 50.34 %
Average organelle coverage : 3910

----------------------------------------------------------------------------------------------------


*** FP
**** 1:10 *worked*
     Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
     Coverage analysis was not selected.  We highly recommend this option to verify the assembly.
**** 1:100 terminated - mem limit 40G ->started again *worked*
Mon Feb 19 09:58:54 2018        Checking chloroplast gene recovery in contigs.
                                81.4814814814815% of known angiosperm chloroplast genes were recovered in fp_sim_1_100_2M_1.fastq_FULLCP.fsa.
Mon Feb 19 09:58:59 2018        Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
                                Coverage analysis was not selected.  We highly recommend this option to verify the assembly.
     
**** 1:1000 terminated - mem limit 40G ->started again *worked*
Mon Feb 19 10:07:01 2018        Checking chloroplast gene recovery in contigs.
                                81.4814814814815% of known angiosperm chloroplast genes were recovered in fp_sim_1_1000_2M_1.fastq_FULLCP.fsa.
Mon Feb 19 10:07:05 2018        Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
                                Coverage analysis was not selected.  We highly recommend this option to verify the assembly.

** DONE Run on ~7 more SSR data sets from 1001 Genom Project
   - State "DONE"       from "PROGRESS"   [2018-02-20 Di 10:12]
   - State "PROGRESS"   from "TODO"       [2018-02-13 Di 17:00]

     
*** CE-befor_fix

**** SRR1945438 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
[18-02-13 16:06:10] [PipeWrap] scr exited:65280 
[18-02-13 16:06:10] [PipeWrap] HASH(0x3e41f10) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.

**** SRR1945439 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
[18-02-14 09:03:33] [PipeWrap] scr exited:65280 
[18-02-14 09:03:33] [PipeWrap] HASH(0x2ca8f80) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.

**** SRR1945440 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
[18-02-14 10:08:54] [PipeWrap] scr exited:65280 
[18-02-14 10:08:54] [PipeWrap] HASH(0x329bd60) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.

**** SRR1945441 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
[18-02-14 12:08:07] [PipeWrap] scr exited:65280 
[18-02-14 12:08:07] [PipeWrap] HASH(0x2a4b350) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.

**** SRR1945442 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
[18-02-14 12:22:21] [PipeWrap] scr exited:65280 
[18-02-14 12:22:21] [PipeWrap] HASH(0x20d51c0) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.

**** SRR1945443 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.

**** SRR1945444 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
**** SRR1945445 *ERROR*
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.

*** CE-fix

**** SRR1945435 *incomplete - partial*
     [18-02-20 06:41:21] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:42:08] [fcg] Found 5 partial chloroplast sequences
     [18-02-20 06:42:09] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:42:09] [PipeWrap] ptx pipeline completed

**** SRR1945436  *incomplete - partial*
     [18-02-20 06:36:41] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:37:03] [fcg] Found 6 partial chloroplast sequences
     [18-02-20 06:37:03] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:37:04] [PipeWrap] ptx pipeline completed
     
**** SRR1945437  *incomplete - partial* 
     [18-02-20 06:36:43] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:37:27] [fcg] Found 5 partial chloroplast sequences
     [18-02-20 06:37:27] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:37:27] [PipeWrap] ptx pipeline completed

**** SRR1945438  *incomplete - partial*
     [18-02-20 06:41:17] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:42:07] [fcg] Found 3 partial chloroplast sequences
     [18-02-20 06:42:07] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:42:07] [PipeWrap] ptx pipeline completed

      #+BEGIN_SRC sh :dir /home/s229502/Master_thesis/runs/SRR1945438/CE/SRR1945438_1.fastq_ce
      SeqFilter fcg.fa

      #+END_SRC 

      #+RESULTS:
      | #source | state | reads |  bases |    max |  min |    N50 |   N90 |
      | fcg.fa  | RAW   |     3 | 127931 | 102250 | 7791 | 102250 | 17890 |

**** SRR1945439  *incomplete - partial* 
     [18-02-20 06:47:53] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:48:33] [fcg] Found 4 partial chloroplast sequences
     [18-02-20 06:48:33] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:48:34] [PipeWrap] ptx pipeline completed

**** SRR1945440  *incomplete - partial*
     [18-02-20 06:49:55] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:50:35] [fcg] Found 4 partial chloroplast sequences
     [18-02-20 06:50:35] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:50:36] [PipeWrap] ptx pipeline completed

**** SRR1945441  *incomplete - partial*
     [18-02-20 06:48:29] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:49:12] [fcg] Found 5 partial chloroplast sequences
     [18-02-20 06:49:12] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:49:13] [PipeWrap] ptx pipeline completed

**** SRR1945442  *incomplete - partial*
     [18-02-20 06:50:42] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:51:18] [fcg] Found 4 partial chloroplast sequences
     [18-02-20 06:51:18] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:51:19] [PipeWrap] ptx pipeline completed
     
**** SRR1945443 *Success*
     [18-02-20 06:51:45] [fcg] Found 3 different nodes
     [18-02-20 06:51:45] [fcg] The inverted repeat is node number: 21
     [18-02-20 06:51:45] [fcg] The LSC is node number: 10 and the SSC is node number: 20
     [18-02-20 06:51:45] [fcg] Single circular chloroplast seems to be found
     [18-02-20 06:51:45] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:51:45] [PipeWrap] ptx pipeline completed

 #+BEGIN_SRC sh :dir /home/s229502/Master_thesis/runs/SRR1945443/CE/SRR1945443_1.fastq_ce
 SeqFilter fcg.fa

 #+END_SRC  

 #+RESULTS:
 | #source | state | reads |  bases |    max |    min |    N50 |    N90 |
 | fcg.fa  | RAW   |     1 | 154679 | 154679 | 154679 | 154679 | 154679 |

**** SRR1945444  *incomplete - partial*
     [18-02-20 06:50:16] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:50:43] [fcg] Found 4 partial chloroplast sequences
     [18-02-20 06:50:43] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:50:43] [PipeWrap] ptx pipeline completed

**** SRR1945445  *incomplete - partial*
     [18-02-20 06:50:09] [fcg] No single circular chloroplast was found. Searching for partial hits...
     [18-02-20 06:50:29] [fcg] Found 4 partial chloroplast sequences
     [18-02-20 06:50:29] [PipeWrap] Running 'mre1': gs -dBATCH -dNOPAUSE -q -sDEVICE=pdfwrite -sOutputFile=ptx.pdf scr-seeds.pdf kfr.pdf
     [18-02-20 06:50:30] [PipeWrap] ptx pipeline completed


*** GO
**** SRR1945438 *ERROR*
2018-02-13 16:08:20,125 - INFO: Assembling with SPAdes ...
2018-02-13 16:08:21,911 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945438/SRR1945438_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-13 16:08:21,913 - ERROR: Assembling failed.

**** SRR1945439 *ERROR*
2018-02-14 09:03:58,070 - INFO: Assembling with SPAdes ...
2018-02-14 09:04:00,235 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945439/SRR1945439_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 09:04:00,237 - ERROR: Assembling failed.

**** SRR1945440 *ERROR*
2018-02-14 10:09:50,316 - INFO: Assembling with SPAdes ...
2018-02-14 10:09:51,960 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945440/SRR1945440_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 10:09:51,962 - ERROR: Assembling failed.

**** SRR1945441 *ERROR*
     2018-02-14 12:09:07,830 - ERROR: Error in SPAdes: 
     == Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945441/SRR1945441_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 12:09:07,832 - ERROR: Assembling failed.

**** SRR1945442 *ERROR*
2018-02-14 12:22:46,623 - INFO: Assembling with SPAdes ...
2018-02-14 12:22:48,751 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945442/SRR1945442_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 12:22:48,753 - ERROR: Assembling failed.

**** SRR1945443 *ERROR*
2018-02-14 12:27:09,501 - INFO: Assembling with SPAdes ...
2018-02-14 12:27:11,117 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945443/SRR1945443_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 12:27:11,119 - ERROR: Assembling failed.

**** SRR1945444 *ERROR*
2018-02-14 12:28:13,692 - INFO: Assembling with SPAdes ...
2018-02-14 12:28:16,411 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945444/SRR1945444_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 12:28:16,412 - ERROR: Assembling failed.

**** SRR1945445 *ERROR*
2018-02-14 12:29:01,280 - ERROR: Error in SPAdes: 
== Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945445/SRR1945445_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255


2018-02-14 12:29:01,283 - ERROR: Assembling failed.

*** GO_Rerun
**** SRR1945435 *finished - F?*
     2018-02-20 13:56:03,143 - INFO: Separating filtered fastq file ... 
     2018-02-20 13:56:07,828 - INFO: Separating filtered fastq file finished!
     2018-02-20 13:56:07,883 - INFO: Assembling with SPAdes ...
     2018-02-20 14:01:01,372 - INFO: Assembling finished.
     
     2018-02-20 14:01:03,231 - INFO: Processing assembly result finished!
     
     Total Calc-cost 809.17240715
     Thanks you!
**** SRR1945436 *finished*
     2018-02-20 14:25:24,331 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:25:32,137 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:25:32,193 - INFO: Assembling with SPAdes ...
     2018-02-20 14:33:56,195 - INFO: Assembling finished.
     
     2018-02-20 14:33:58,579 - INFO: Processing assembly result finished!

     Total Calc-cost 1037.79737711
     Thanks you!
**** SRR1945437 *finished* 
     2018-02-20 14:25:47,834 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:25:55,800 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:25:55,898 - INFO: Assembling with SPAdes ...
     2018-02-20 14:34:16,099 - INFO: Assembling finished.
     
     2018-02-20 14:34:18,821 - INFO: Processing assembly result finished!
     
     Total Calc-cost 1044.55838513
     Thanks you!

**** SRR1945438 *finished* 
     2018-02-20 14:26:12,765 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:26:18,611 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:26:18,667 - INFO: Assembling with SPAdes ...
     2018-02-20 14:34:26,286 - INFO: Assembling finished.
     
     2018-02-20 14:34:29,866 - INFO: Processing assembly result finished!
     
     Total Calc-cost 974.736760139
     Thanks you!
**** SRR1945439 *finished* 
     2018-02-20 14:26:41,118 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:26:47,614 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:26:47,668 - INFO: Assembling with SPAdes ...
     2018-02-20 14:33:25,656 - INFO: Assembling finished.
     
     2018-02-20 14:33:27,286 - INFO: Processing assembly result finished!

     Total Calc-cost 884.311291933
     Thanks you!

**** SRR1945440 *finished*
     2018-02-20 14:27:44,111 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:27:48,091 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:27:48,134 - INFO: Assembling with SPAdes ...
     2018-02-20 14:34:32,126 - INFO: Assembling finished.
     
     2018-02-20 14:34:33,992 - INFO: Processing assembly result finished!

     Total Calc-cost 924.914399147
     Thanks you!

**** SRR1945441 *finished*
     2018-02-20 14:27:48,350 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:27:53,340 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:27:53,387 - INFO: Assembling with SPAdes ...
     2018-02-20 14:35:23,324 - INFO: Assembling finished.
     
     2018-02-20 14:35:25,157 - INFO: Processing assembly result finished!

     Total Calc-cost 963.723965883
     Thanks you!

**** SRR1945442 *finished*
     2018-02-20 14:27:57,905 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:28:02,568 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:28:02,636 - INFO: Assembling with SPAdes ...
     2018-02-20 14:34:43,336 - INFO: Assembling finished.
     
     2018-02-20 14:34:45,402 - INFO: Processing assembly result finished!
     
     Total Calc-cost 911.527772903
     Thanks you!
**** SRR1945443 *finished*
     2018-02-20 14:28:39,985 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:28:43,729 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:28:43,784 - INFO: Assembling with SPAdes ...
     2018-02-20 14:35:16,253 - INFO: Assembling finished.
     
     2018-02-20 14:35:17,927 - INFO: Processing assembly result finished!
     
     Total Calc-cost 932.15546298
     Thanks you!
**** SRR1945444 *Finished*
     2018-02-20 14:28:30,443 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:28:35,529 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:28:35,695 - INFO: Assembling with SPAdes ...
     2018-02-20 14:35:43,619 - INFO: Assembling finished.

     2018-02-20 14:35:45,335 - INFO: Processing assembly result finished!
     
     Total Calc-cost 951.788398027
     Thanks you!

**** SRR1945445 *Finished*
     2018-02-20 14:28:26,796 - INFO: Separating filtered fastq file ... 
     2018-02-20 14:28:31,856 - INFO: Separating filtered fastq file finished!
     2018-02-20 14:28:32,006 - INFO: Assembling with SPAdes ...
     2018-02-20 14:36:02,592 - INFO: Assembling finished.
     
     2018-02-20 14:36:04,311 - INFO: Processing assembly result finished!
     
     Total Calc-cost 961.558608055
     Thanks you!


*** IOGA
**** SRR1945435 *worked - 6contigs*
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/FirstTest_9-2/IOGA_9-2/SRR1945435/SRR1945435_1.fastq_IOGA_OUT.final/SRR1945435_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945435_1.fastq_IOGA_OUT.spades.k.55.77.95.contigs  -5688968.210842 6 129055 1 66884 has the highest ALE score
     [IOGA] Started  09:21:03.204548
     [IOGA] Finished 21:05:41.883040
     
**** SRR1945436 *worked - 8 contigs*
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/FirstTest_9-2/IOGA_9-2/SRR1945436/SRR1945436_1.fastq_IOGA_OUT.final/SRR1945436_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945436_1.fastq_IOGA_OUT.spades.k.55.77.95.contigs  -5514558.659379 8 128290 1 79142 has the highest ALE score
     [IOGA] Started  09:21:54.740627
     [IOGA] Finished 21:38:45.747710

**** SRR1945437 *Worked - 10 contigs*
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/FirstTest_9-2/IOGA_9-2/SRR1945437/SRR1945437_1.fastq_IOGA_OUT.final/SRR1945437_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945437_1.fastq_IOGA_OUT.spades.k.77.scaffolds  -5644781.995918 10 129141 2 22943 has the highest ALE score
     [IOGA] Started  09:22:33.546328
     [IOGA] Finished 22:26:27.535550

**** SRR1945438 ERROR - again DIR -> *worked - 10 contigs*
# Traceback (most recent call last):
#  File "/IOGA/IOGA.py", line 432, in <module>
#    main(args.reference, args.name, args.forward, args.reverse, args.threads, args.insertsize, args.maxrounds, args.verbose)
#  File "/IOGA/IOGA.py", line 377, in main
#    source,FP,RP,final_iteration = IOGA_loop(name,ref,forward,reverse,insertsize,threads,maxrounds)
#  File "/IOGA/IOGA.py", line 322, in IOGA_loop
#    shutil.rmtree(name + '.' + str(iteration-2))
#  File "/usr/lib/python2.7/shutil.py", line 239, in rmtree
#    onerror(os.listdir, path, sys.exc_info())
#  File "/usr/lib/python2.7/shutil.py", line 237, in rmtree
#    names = os.listdir(path)
# OSError: [Errno 2] No such file or directory: '../SRR1945438_1.fastq_IOGA_OUT.1'


     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/runs/SRR1945438/SRR1945438_1.fastq_IOGA_OUT.final/SRR1945438_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945438_1.fastq_IOGA_OUT.spades.k.55.77.95.scaffolds  -4552290.456234 10 113515 1 58487 has the highest ALE score
     [IOGA] Started  09:08:21.709022
     [IOGA] Finished 13:59:25.322277

**** SRR1945439 Terminated - more than 16G -> Startet with 25G again -> with 30G again -> with 50G again *Worked - 10 contigs*
   #  slurmstepd: Job 31354 exceeded memory limit (18092360 > 16777216), being killed
   #  slurmstepd: Exceeded job memory limit
   #  slurmstepd: *** JOB 31354 ON uranus1 CANCELLED AT 2018-02-15T02:21:41 ***

    # Docker image path: index.docker.io/chloroextractorteam/ioga_docker:latest
    # Cache folder set to /home/s229502/.singularity/docker
    # Creating container runtime...
    # slurmstepd: Job 31633 exceeded memory limit (32557380 > 31457280), being killed
    # slurmstepd: Exceeded job memory limit
    # slurmstepd: *** JOB 31633 ON jupiter CANCELLED AT 2018-02-17T12:03:02 ***
      
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/runs/SRR1945439/IOGA/SRR1945439_1.fastq_IOGA_OUT.final/SRR1945439_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945439_1.fastq_IOGA_OUT.spades.k.77.scaffolds  -5414517.025604 10 106360 2 22912 has the highest ALE score
     [IOGA] Started  08:49:31.209760
     [IOGA] Finished 18:47:57.070384

**** SRR1945440 *worked - 9 contigs*
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/runs/SRR1945440/SRR1945440_1.fastq_IOGA_OUT.final/SRR1945440_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945440_1.fastq_IOGA_OUT.spades.k.77.scaffolds  -4582014.048077 9 108509 2 30328 has the highest ALE score
     [IOGA] Started  10:07:20.650823
     [IOGA] Finished 14:47:27.054108

**** SRR1945441 *worked - 6 contigs*
     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/runs/SRR1945441/SRR1945441_1.fastq_IOGA_OUT.final/SRR1945441_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945441_1.fastq_IOGA_OUT.spades.k.55.77.95.scaffolds  -5133410.247351 6 110704 1 76914 has the highest ALE score
     [IOGA] Started  12:08:02.628599
     [IOGA] Finished 21:09:33.686531
**** SRR1945442
**** SRR1945443 Terminated - memory -> startet with 25G again *worked - 8 contis*
     
     # Creating container runtime...
     # slurmstepd: Job 31438 exceeded memory limit (18430644 > 16777216), being killed
     # slurmstepd: Exceeded job memory limit
     # slurmstepd: *** JOB 31438 ON uranus1 CANCELLED AT 2018-02-15T07:05:40 ***


     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945444_1.fastq_IOGA_OUT.spades.k.55.77.95.contigs  -4670142.974645 8 109521 1 79118 has the highest ALE score
     [IOGA] Started  09:26:02.588976
     [IOGA] Finished 03:46:16.606412

**** SRR1945444 Terminated - memory -> startet with 25G again *worked - 8 contigs*
     # Creating container runtime...
     # slurmstepd: Job 31439 exceeded memory limit (16796576 > 16777216), being killed
     # slurmstepd: Exceeded job memory limit
     # slurmstepd: *** JOB 31439 ON uranus1 CANCELLED AT 2018-02-15T04:42:38 ***

     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/runs/SRR1945444/SRR1945444_1.fastq_IOGA_OUT.final/SRR1945444_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945444_1.fastq_IOGA_OUT.spades.k.55.77.95.contigs  -4670142.974645 8 109521 1 79118 has the highest ALE score
     [IOGA] Started  09:26:02.588976
     [IOGA] Finished 03:46:16.606412

**** SRR1945445 Terminated - memory -> startet with 25G again *worked - 7 contigs*
    # slurmstepd: Job 31437 exceeded memory limit (18503268 > 16777216), being killed
    # slurmstepd: Exceeded job memory limit
    # slurmstepd: *** JOB 31437 ON uranus1 CANCELLED AT 2018-02-15T06:16:03 ***


     [IOGA] final assembly statistics are in /home/s229502/Master_thesis/runs/SRR1945445/IOGA/SRR1945445_1.fastq_IOGA_OUT.final/SRR1945445_1.fastq_IOGA_OUT.statistics
     [IOGA] Filename ALE_score Contig_no Assembly_size N50_index N50
     [IOGA] SRR1945445_1.fastq_IOGA_OUT.spades.k.55.77.95.contigs  -5161440.480749 7 110817 1 102352 has the highest ALE score
     [IOGA] Started  12:25:28.043213
     [IOGA] Finished 18:40:28.950049


*** O.A
**** SRR1945438 *ERROR ?*
Reading indexed sequence reads...

        0 sequences read

Reading indexed pair data...

Done.

Loading reverse index...

Done.

Indexing reverse complement sequences ...


Fast indexing forward reads...


Fast indexing reverse reads...

Done.
2018-02-13 16:15:16,777 [INFO ]  No new probe set specified
2018-02-13 16:15:16,778 [INFO ]  No new probe set specified
2018-02-13 16:15:16,778 [INFO ]  No previous matches loaded
2018-02-13 16:15:16,779 [INFO ]  No --seeds option specified and not previous matches stored

**** SRR1945439 *ERROR - by connecting..*
2018-02-14 10:32:12,590 [INFO ]  Minimum stem coverage = 41
2018-02-14 10:32:20,023 [INFO ]  -19 -> -19 scaffolded by 393 pair ended links
2018-02-14 10:32:20,033 [INFO ]  -19 -> 15 overlap of 49bp supported by 8 pair ended links
2018-02-14 10:32:20,447 [INFO ]  -15 -> 19 overlap of 49bp supported by 8 pair ended links
2018-02-14 10:32:20,563 [INFO ]  -2 -> -19 overlap of 18bp supported by 133 pair ended links
2018-02-14 10:32:20,750 [INFO ]  19 -> 2 overlap of 18bp supported by 133 pair ended links
2018-02-14 10:32:20,801 [INFO ]  19 -> 19 scaffolded by 393 pair ended links
2018-02-14 10:32:21,037 [INFO ]  Select the good connected components
Traceback (most recent call last):
  File "/usr/bin/oa", line 64, in <module>
    config[root_config_name]['module'].run(config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 194, in run
    paths=buildPath(asm, cg, back, minlink, config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 113, in buildPath
    back=int(back),minlink=minlink)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 1665, in pathConstraints
    constraints = mergeConstraints(constraints,minisatDoubleConstraints(self,cg))
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 1532, in minisatDoubleConstraints
    r1 = round(middleFcov/float(inputcov))
ZeroDivisionError: float division by zero

**** SRR1945440 *ERROR*
Loading global data...

Done.

Reading indexed sequence reads...

        0 sequences read

Reading indexed pair data...

Done.

Loading reverse index...

Done.

Indexing reverse complement sequences ...


Fast indexing forward reads...


Fast indexing reverse reads...

Done.
2018-02-14 10:18:12,406 [INFO ]  No new probe set specified
2018-02-14 10:18:12,407 [INFO ]  No new probe set specified
2018-02-14 10:18:12,407 [INFO ]  No previous matches loaded
2018-02-14 10:18:12,408 [INFO ]  No --seeds option specified and not previous matches stored

**** SRR1945441 *worked - linear*
2018-02-14 14:23:17,823 [INFO ]  Minimum stem coverage = 260
2018-02-14 14:23:23,575 [INFO ]  Both segments 4 and -1 are disconnected
2018-02-14 14:23:23,577 [INFO ]     But overlap by 43bp supported by 6 pair ended links
2018-02-14 14:23:23,596 [INFO ]  Both segments -1 and 2 are disconnected
2018-02-14 14:23:23,597 [INFO ]     But overlap by 58bp supported by 319 pair ended links
2018-02-14 14:23:23,638 [INFO ]  Both segments 2 and -3 are connected (paired-end=179 frg length=284.575419 sd=84.065442)
2018-02-14 14:23:23,688 [INFO ]  Both segments -3 and -2 are connected (paired-end=364 frg length=280.167582 sd=87.554891)
2018-02-14 14:23:23,706 [INFO ]  Both segments -2 and 1 are disconnected
2018-02-14 14:23:23,707 [INFO ]     But overlap by 58bp supported by 319 pair ended links
2018-02-14 14:23:23,725 [INFO ]  Both segments 1 and -4 are disconnected
2018-02-14 14:23:23,725 [INFO ]     But overlap by 43bp supported by 6 pair ended links
2018-02-14 14:23:23,732 [INFO ]  Path is linear

**** SRR1945442 *ERROR*
Traceback (most recent call last):
  File "/usr/bin/oa", line 64, in <module>
    config[root_config_name]['module'].run(config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 149, in run
    asm = restoreGraph(output+'.oax',r,x)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 2068, in restoreGraph
    filein = open(filein)
FileNotFoundError: [Errno 2] No such file or directory: '../SRR1945442_1.fastq_oa_test.chl.oas/assembling.oax'

**** SRR1945443 *worked - linear*
2018-02-14 13:26:13,121 [INFO ]  Minimum stem coverage = 17
2018-02-14 13:26:16,321 [INFO ]  Both segments -2 and -13 are disconnected
2018-02-14 13:26:16,322 [INFO ]     But linked by 211 pair ended links (gap length=20.000000 sd=103.000000)
2018-02-14 13:26:16,342 [INFO ]  Both segments -13 and -1 are connected (paired-end=254 frg length=362.015748 sd=100.603435)
2018-02-14 13:26:16,351 [INFO ]  Both segments -1 and -11 are connected (paired-end=75 frg length=303.640000 sd=86.607009)
2018-02-14 13:26:16,361 [INFO ]  Both segments -11 and -5 are connected (paired-end=120 frg length=292.466667 sd=118.460144)
2018-02-14 13:26:16,397 [INFO ]  Both segments -5 and -14 are connected (paired-end=337 frg length=335.937685 sd=110.606602)
2018-02-14 13:26:16,422 [INFO ]  Both segments -14 and 5 are connected (paired-end=178 frg length=328.382022 sd=103.439767)
2018-02-14 13:26:16,444 [INFO ]  Path is linear

**** SRR1945444 *worked - linear*
2018-02-14 13:44:12,715 [INFO ]  Expanded path : (12,)
2018-02-14 13:44:14,272 [INFO ]  Compacting graph  19.8 % remain : 00:00:06
2018-02-14 13:44:14,403 [INFO ]  Compacting graph  28.8 % remain : 00:00:04
2018-02-14 13:44:14,554 [INFO ]  Compacting graph  40.6 % remain : 00:00:02
2018-02-14 13:44:14,707 [INFO ]  Compacting graph  51.9 % remain : 00:00:01
2018-02-14 13:44:14,882 [INFO ]  Compacting graph  65.4 % remain : 00:00:01
2018-02-14 13:44:15,356 [INFO ]  Compacting graph  96.9 % remain : 00:00:00
2018-02-14 13:44:15,448 [INFO ]  Minimum stem coverage = 51
2018-02-14 13:44:19,793 [INFO ]  Path is linear

**** SRR1945445 *ERROR*
2018-02-14 12:34:06,459 [INFO ]  No new probe set specified
2018-02-14 12:34:06,460 [INFO ]  No new probe set specified
2018-02-14 12:34:06,461 [INFO ]  No previous matches loaded
2018-02-14 12:34:06,461 [INFO ]  No --seeds option specified and not previous matches stored

*** O.A_Rerun
**** SRR1945435 *ERROR*
     2018-02-22 01:26:04,323 [INFO ]  44 -> 44 scaffolded by 530 pair ended links
     2018-02-22 01:26:05,264 [INFO ]  -36 -> -20 scaffolded by 290 pair ended links
     2018-02-22 01:26:05,827 [INFO ]  -36 -> -36 scaffolded by 43 pair ended links
     2018-02-22 01:26:06,050 [INFO ]  -70 -> 48 scaffolded by 17 pair ended links
     2018-02-22 01:26:06,544 [INFO ]  -70 -> -29 scaffolded by 131 pair ended links
     2018-02-22 01:26:07,530 [INFO ]  -70 -> -70 scaffolded by 112 pair ended links
     2018-02-22 01:26:08,157 [INFO ]  65 -> 18 scaffolded by 90 pair ended links
     2018-02-22 01:26:08,810 [INFO ]  65 -> 65 scaffolded by 15 pair ended links
     2018-02-22 01:26:10,346 [INFO ]  57 -> 21 overlap of 51bp supported by 520 pair ended links
     2018-02-22 01:26:10,446 [INFO ]  57 -> 57 scaffolded by 228 pair ended links
     2018-02-22 01:26:11,257 [INFO ]  67 -> 80 overlap of 51bp supported by 453 pair ended links
     2018-02-22 01:26:12,471 [INFO ]  67 -> 67 scaffolded by 495 pair ended links
     2018-02-22 01:26:13,792 [INFO ]  62 -> 62 scaffolded by 72 pair ended links
     2018-02-22 01:26:13,879 [INFO ]  Select the good connected components
Traceback (most recent call last):
  File "/usr/bin/oa", line 64, in <module>
    config[root_config_name]['module'].run(config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 194, in run
    paths=buildPath(asm, cg, back, minlink, config)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/commands/unfold.py", line 113, in buildPath
    back=int(back),minlink=minlink)
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 1665, in pathConstraints
    constraints = mergeConstraints(constraints,minisatDoubleConstraints(self,cg))
  File "/org-asm/ORG.asm-1.0.00-alpha11/lib/python3.5/site-packages/orgasm/tango.py", line 1532, in minisatDoubleConstraints
    r1 = round(middleFcov/float(inputcov))
ZeroDivisionError: float division by zero

**** SRR1945436 *Incomplete - linear*
     2018-02-20 10:12:25,770 [INFO ]  Compacting graph  14.0 % remain : 00:00:08
     2018-02-20 10:12:26,215 [INFO ]  Compacting graph  46.0 % remain : 00:00:02
     2018-02-20 10:12:26,368 [INFO ]  Compacting graph  57.0 % remain : 00:00:01
     2018-02-20 10:12:26,772 [INFO ]  Compacting graph  85.9 % remain : 00:00:00
     2018-02-20 10:12:26,924 [INFO ]  Compacting graph  96.9 % remain : 00:00:00
     2018-02-20 10:12:28,798 [INFO ]  Minimum stem coverage = 330
     2018-02-20 10:12:34,757 [INFO ]  Path is linear

**** SRR1945437 *Incomplete - linear*
     2018-02-20 10:05:32,968 [INFO ]  Expanded path : (7,)
     2018-02-20 10:05:34,313 [INFO ]  Compacting graph  12.6 % remain : 00:00:09
     2018-02-20 10:05:34,471 [INFO ]  Compacting graph  24.3 % remain : 00:00:04
     2018-02-20 10:05:34,768 [INFO ]  Compacting graph  46.4 % remain : 00:00:02
     2018-02-20 10:05:34,991 [INFO ]  Compacting graph  62.7 % remain : 00:00:01
     2018-02-20 10:05:35,264 [INFO ]  Compacting graph  83.0 % remain : 00:00:00
     2018-02-20 10:05:37,221 [INFO ]  Minimum stem coverage = 47
     2018-02-20 10:05:40,656 [INFO ]  Path is linear

**** SRR1945438 *FAIL*
     2018-02-20 09:15:19,709 [INFO ]  No new probe set specified
     2018-02-20 09:15:19,710 [INFO ]  No new probe set specified
     2018-02-20 09:15:19,710 [INFO ]  No previous matches loaded
     2018-02-20 09:15:19,710 [INFO ]  No --seeds option specified and not previous matches stored
     
**** SRR1945439 *Incomplete - linear*
     2018-02-20 11:24:33,766 [INFO ]  Minimum stem coverage = 91
     2018-02-20 11:24:38,143 [INFO ]  Both segments 10 and 6 are disconnected
     2018-02-20 11:24:38,144 [INFO ]     But overlap by 49bp supported by 8 pair ended links
     2018-02-20 11:24:38,162 [INFO ]  Path is linea
**** SRR1945440 *FAIL*
     
     2018-02-20 09:17:18,436 [INFO ]  No new probe set specified
     2018-02-20 09:17:18,437 [INFO ]  No new probe set specified
     2018-02-20 09:17:18,437 [INFO ]  No previous matches loaded
     2018-02-20 09:17:18,437 [INFO ]  No --seeds option specified and not previous matches stored
**** SRR1945441 *incomplete - linear*
     2018-02-20 10:03:20,106 [INFO ]     But linked by 310 pair ended links (gap length=4.000000 sd=80.000000)
     2018-02-20 10:03:20,125 [INFO ]  Both segments -6 and -4 are connected (paired-end=179 frg length=284.575419 sd=84.065442)
     2018-02-20 10:03:20,140 [INFO ]  Both segments -4 and 6 are connected (paired-end=364 frg length=280.167582 sd=87.554891)
     2018-02-20 10:03:20,147 [INFO ]  Both segments 6 and 2 are disconnected
     2018-02-20 10:03:20,147 [INFO ]     But linked by 310 pair ended links (gap length=4.000000 sd=80.000000)
     2018-02-20 10:03:20,161 [INFO ]  Path is linear

**** SRR1945442      
**** SRR1945443 *incomplete - linear*
     2018-02-20 09:57:42,548 [INFO ]  Both segments 7 and 6 are connected (paired-end=133 frg length=267.766917 sd=76.915133)
     2018-02-20 09:57:42,551 [INFO ]  Both segments 6 and -13 are disconnected
     2018-02-20 09:57:42,551 [INFO ]     But overlap by 71bp supported by 24 pair ended links
     2018-02-20 09:57:42,556 [INFO ]  Path is linear

**** SRR1945444 *incomplete- linear*
     2018-02-20 12:10:21,349 [INFO ]  Compacting graph  34.6 % remain : 00:00:03
     2018-02-20 12:10:21,500 [INFO ]  Compacting graph  45.9 % remain : 00:00:02
     2018-02-20 12:10:21,991 [INFO ]  Compacting graph  78.7 % remain : 00:00:00
     2018-02-20 12:10:22,160 [INFO ]  Compacting graph  90.0 % remain : 00:00:00
     2018-02-20 12:10:22,197 [INFO ]  Compacting graph  91.7 % remain : 00:00:00
     2018-02-20 12:10:24,219 [INFO ]  Minimum stem coverage = 286
     2018-02-20 12:10:28,640 [INFO ]  Path is linear

**** SRR1945445 *FAIL*
2018-02-20 09:19:35,846 [INFO ]  No new probe set specified
2018-02-20 09:19:35,847 [INFO ]  No new probe set specified
2018-02-20 09:19:35,848 [INFO ]  No previous matches loaded
2018-02-20 09:19:35,848 [INFO ]  No --seeds option specified and not previous matches stored
 
    
*** NP_2.6.3
**** SRR1945438 *ERROR - seed*
     Building Hash Table......OK

     Subsampled fraction: 100.00 %

     Retrieve Seed...

     INVALID SEED, PLEASE TRY AGAIN WITH A NEW ONE
**** SRR1945439 *ERROR*
Building Hash Table......OK

Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.

**** SRR1945440 *ERROR - seed*
Retrieve Seed...

INVALID SEED, PLEASE TRY AGAIN WITH A NEW ONE

**** SRR1945441 *ERROR*
Building Hash Table......OK

Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.

**** SRR1945442 *ERROR*
Building Hash Table......OK

Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.

**** SRR1945443 *ERROR*
Building Hash Table......OK

Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.

**** SRR1945444 *ERROR*
Building Hash Table......OK

Illegal division by zero at /NOVOPlasty/NOVOPlasty2.6.3.pl line 2045.
**** SRR1945445 *ERROR - seed*
Retrieve Seed...

INVALID SEED, PLEASE TRY AGAIN WITH A NEW ONE

*** NP_2.6.4
**** SRR1945435 *incomplete - 4 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------

     Contig 01                  : 146094 bp
     Contig 02                  : 407 bp
     Estimated gap      : 0 bp to 0 bp
     Contig 02                  : 8047 bp
     Estimated gap      : 0 bp to 0 bp
     Contig 02                  : 1012 bp
     
     Total contigs              : 4
     Largest contig             : 146094 bp
     Smallest contig            : 407 bp
     Average insert size        : 252 bp
     
     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 624986
     Assembled reads            : 208354
     Organelle genome %         : 15.62 %
     Average organelle coverage : 405
     
     ----------------------------------------------------------------------------------------------------
     
     
     Thank you for using NOVOPlasty!
**** SRR1945436 *incomplete - 3 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------
     
     Contig 01                  : 166043 bp
     Estimated gap      : -17 bp to -17 bp (Check manually if the two contigs overlap to merge them together!)
     Contig 01                  : 6077 bp
     Estimated gap      : -20 bp to -20 bp (Check manually if the two contigs overlap to merge them together!)
     Contig 01                  : 22149 bp
     
     Total contigs              : 3
     Largest contig             : 166043 bp
     Smallest contig            : 6077 bp
     Average insert size        : 254 bp
     
     
     -----------------------------------------Input data metrics-----------------------------------------

     Total reads                : 4000000
     Aligned reads              : 831154
     Assembled reads            : 255604
     Average organelle coverage : 428
     
     ----------------------------------------------------------------------------------------------------


     Thank you for using NOVOPlasty!
     
**** SRR1945437 *incomplete - 4 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------
     
     Contig 01                  : 118180 bp
     Contig 02                  : 374 bp
     Estimated gap      : 0 bp to 0 bp
     Contig 02                  : 30191 bp
     Estimated gap      : 0 bp to 0 bp
     Contig 02                  : 6560 bp

     Total contigs              : 4
     Largest contig             : 118180 bp
     Smallest contig            : 374 bp
     Average insert size        : 247 bp
     

     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 875718
     Assembled reads            : 259182
     Organelle genome %         : 21.89 %
     Average organelle coverage : 567

     ----------------------------------------------------------------------------------------------------
     
     
     Thank you for using NOVOPlasty!
**** SRR1945438 *incomplete - 6 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------

     Contig 01                  : 30860 bp
     Contig 02                  : 254 bp
     Estimated gap      : -11 bp to -11 bp (Check manually if the two contigs overlap to merge them together!)
     Contig 02                  : 87305 bp
     Contig 03                  : 8328 bp
     Estimated gap      : -11 bp to -11 bp (Check manually if the two contigs overlap to merge them together!)
     Contig 03                  : 28270 bp
     Estimated gap      : -36 bp to -36 bp (Check manually if the two contigs overlap to merge them together!)
     Contig 03                  : 1051 bp
     
     Total contigs              : 6
     Largest contig             : 87305 bp
     Smallest contig            : 254 bp
     Average insert size        : 244 bp
     

     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 718126
     Assembled reads            : 227078
     Organelle genome %         : 17.95 %
     Average organelle coverage : 461
     
     ----------------------------------------------------------------------------------------------------


     Thank you for using NOVOPlasty!

**** SRR1945439 *1 contig*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------
     
     Contig 01                  : 74084 bp
     
     Total contigs              : 1
     Largest contig             : 74084 bp
     Smallest contig            : 74084 bp
     Average insert size        : 217 bp
     
     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 682198
     Assembled reads            : 245142
     Average organelle coverage : 921
     
     ----------------------------------------------------------------------------------------------------

     
     Thank you for using NOVOPlasty!
**** SRR1945440 *FAIL - 0 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------
     
     
     Total contigs              : 0
     Largest contig             : 0 bp
     Smallest contig            : 1000000000000000000000000000000000 bp
     Average insert size        : 300 bp

     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 300
     Assembled reads            : 0
     
     ----------------------------------------------------------------------------------------------------

     
     Thank you for using NOVOPlasty!
     
**** SRR1945441 *Fail - 0 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------
     
     
     Total contigs              : 0
     Largest contig             : 0 bp
     Smallest contig            : 1000000000000000000000000000000000 bp
     Average insert size        : 300 bp
     
     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 404
     Assembled reads            : 0
     
     ----------------------------------------------------------------------------------------------------
     
     
     Thank you for using NOVOPlasty!

**** SRR1945442 *1 contig*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------
     
     Contig 01                  : 200011 bp
     
     Total contigs              : 1
     Largest contig             : 200011 bp
     Smallest contig            : 200011 bp
     Average insert size        : 244 bp
     
     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 738286
     Assembled reads            : 242504
     Average organelle coverage : 369

     ----------------------------------------------------------------------------------------------------

     
     Thank you for using NOVOPlasty!
**** SRR1945443 *incomplete - 2 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------
     
     Contig 01                  : 82349 bp
     Contig 02                  : 72971 bp
     
     Total contigs              : 2
     Largest contig             : 82349 bp
     Smallest contig            : 72971 bp
     Average insert size        : 292 bp

     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 643478
     Assembled reads            : 186282
     Organelle genome %         : 16.09 %
     Average organelle coverage : 417

     ----------------------------------------------------------------------------------------------------
     

     Thank you for using NOVOPlasty!
**** SRR1945444 *FAIL - 0 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------

     
     Total contigs              : 0
     Largest contig             : 0 bp
     Smallest contig            : 1000000000000000000000000000000000 bp
     Average insert size        : 300 bp
     
     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 418
     Assembled reads            : 0
     
     ----------------------------------------------------------------------------------------------------


     Thank you for using NOVOPlasty!
     
**** SRR1945445 *Fail - 0 contigs*
     ------------Assembly 1 finished: Contigs are automatically merged in Merged_contigs file------------

     
     Total contigs              : 0
     Largest contig             : 0 bp
     Smallest contig            : 1000000000000000000000000000000000 bp
     Average insert size        : 300 bp
     
     
     -----------------------------------------Input data metrics-----------------------------------------
     
     Total reads                : 4000000
     Aligned reads              : 468
     Assembled reads            : 6
     
     ----------------------------------------------------------------------------------------------------


     Thank you for using NOVOPlasty!



*** FP
**** SRR1945435 *incomplete*
Checking chloroplast gene recovery in contigs.
Checking coverage of final assembly. Final assembly is the last afin iteration.
71.6049382716049% of known angiosperm chloroplast genes were recovered in fp_SRR1945435_1.fastq_afin_iter2.fa.
Could not properly orientate the plastome. Either your plastome does not have an IR or there was an issue with the assembly. Best contigs are in /home/s229502/Master_thesis/runs/SRR1945435/FP/fp_SRR1945435_1.fastq/Final_Assembly/fp_SRR1945435_1.fastq_afin_iter2.fa. A list of genes in each contig can be found in "Chloroplast_gene_composition_of_final_contigs.txt".

**** SRR1945436 Terminated - mem limit 35G -> started again -> *worked*
     Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
     Coverage analysis was not selected.  We highly recommend this option to verify the assembly.
**** SRR1945437 Terminated - mem limit 35G -> started again -> *incomplete*
     Checking chloroplast gene recovery in contigs.
     Checking coverage of final assembly. Final assembly is the last afin iteration.
     56.7901234567901% of known angiosperm chloroplast genes were recovered in fp_SRR1945437_1.fastq_afin_iter2.fa.
     Could not properly orientate the plastome. Either your plastome does not have an IR or there was an issue with the assembly. 
     Best contigs are in /home/s229502/Master_thesis/runs/SRR1945437/FP/fp_SRR1945437_1.fastq/Final_Assembly/fp_SRR1945437_1.fastq_afin_iter2.fa. A list of genes in each contig can be found in "Chloroplast_gene_composition_of_final_contigs.txt".

**** SRR1945438 *worked*
     Checking chloroplast gene recovery in contigs.
     80.2469135802469% of known angiosperm chloroplast genes were recovered in fp_SRR1945438_1.fastq_FULLCP.fsa.
     Fri Feb 16 12:33:01 2018        Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
     Coverage analysis was not selected.  We highly recommend this option to verify the assembly.

**** SRR1945439 Terminated - mem limit 35G -> started again *Worked*
     Mon Feb 19 09:45:50 2018        Checking chloroplast gene recovery in contigs.
                                      81.4814814814815% of known angiosperm chloroplast genes were recovered in fp_SRR1945439_1.fastq_FULLCP.fsa.
     Mon Feb 19 09:45:55 2018        Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
                                     Coverage analysis was not selected.  We highly recommend this option to verify the assembly.
**** SRR1945440 Terminated - mem limit 35G -> started again *worked* 
                                     81.4814814814815% of known angiosperm chloroplast genes were recovered in fp_SRR1945440_1.fastq_FULLCP.fsa.
     Mon Feb 19 09:47:27 2018        Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
                                     Coverage analysis was not selected.  We highly recommend this option to verify the assembly.

**** SRR1945441 Terminated - mem limit 35G -> started again *Worked*
     Mon Feb 19 09:50:52 2018        Checking chloroplast gene recovery in contigs.
                                81.4814814814815% of known angiosperm chloroplast genes were recovered in fp_SRR1945441_1.fastq_FULLCP.fsa.
     Mon Feb 19 09:50:56 2018        Assembly finished. Check Final_Assembly directory for chloroplast assembly and accessory files.
                                Coverage analysis was not selected.  We highly recommend this option to verify the assembly.
     
**** SRR1945442 Terminated - mem limit 35G -> started again *Incomplete*
     Mon Feb 19 09:57:36 2018        Checking chloroplast gene recovery in contigs.
                                Checking coverage of final assembly. Final assembly is the last afin iteration.
                                81.4814814814815% of known angiosperm chloroplast genes were recovered in fp_SRR1945442_1.fastq_afin_iter2.fa.
                                Could not properly orientate the plastome. Either your plastome does not have an IR or there was an issue with the assembly. 
				Best contigs are in /home/s229502/Master_thesis/runs/SRR1945442/FP/fp_SRR1945442_1.fastq/Final_Assembly/fp_SRR1945442_1.fastq_afin_iter2.fa. A list of genes in each contig can be found in "Chloroplast_gene_composition_of_final_contigs.txt".

**** SRR1945443 Terminated - mem limit 35G -> started again *Incomplete*
     Mon Feb 19 09:56:56 2018        Checking chloroplast gene recovery in contigs.
                                Checking coverage of final assembly. Final assembly is the last afin iteration.
                                77.7777777777778% of known angiosperm chloroplast genes were recovered in fp_SRR1945443_1.fastq_afin_iter2.fa.
                                Could not properly orientate the plastome. Either your plastome does not have an IR or there was an issue with the assembly. 
				Best contigs are in /home/s229502/Master_thesis/runs/SRR1945443/FP/fp_SRR1945443_1.fastq/Final_Assembly/fp_SRR1945443_1.fastq_afin_iter2.fa. A list of genes in each contig can be found in "Chloroplast_gene_composition_of_final_contigs.txt".

**** SRR1945444 *Incomplete*
 Checking chloroplast gene recovery in contigs.
 Checking coverage of final assembly. Final assembly is the last afin iteration.
 65.4320987654321% of known angiosperm chloroplast genes were recovered in fp_SRR1945444_1.fastq_afin_iter2.fa.
 Could not properly orientate the plastome. Either your plastome does not have an IR or there was an issue with the assembly. Best contigs are in /home/s229502/Master_thesis/runs/SRR1945444/FP/fp_SRR1945444_1.fastq/Final_Assembly/fp_SRR1945444_1.fastq_afin_iter2.fa. A list of genes in each contig can be found in "Chloroplast_gene_composition_of_final_contigs.txt".

**** SRR1945445 *INcomplete*
     Checking chloroplast gene recovery in contigs.
     Checking coverage of final assembly. Final assembly is the last afin iteration.
     70.3703703703704% of known angiosperm chloroplast genes were recovered in fp_SRR1945445_1.fastq_afin_iter2.fa.
     Could not properly orientate the plastome. Either your plastome does not have an IR or there was an issue with the assembly. Best contigs are in /home/s229502/Master_thesis/runs/SRR1945445/FP/fp_SRR1945445_1.fastq/Final_Assembly/fp_SRR1945445_1.fastq_afin_iter2.fa. A list of genes in each contig can be found in "Chloroplast_gene_composition_of_final_contigs.txt".

     

** DONE Fixing chloroExtractor bug (scale_reads)
     - State "DONE"       from "PROGRESS"   [2018-02-15 Do 15:11]
     - State "PROGRESS"   from "TODO"       [2018-02-15 Do 10:02]
Reading seed kmers
Reading total kmers
Plotting seed and total kmers
Use of uninitialized value $seqwithtlen in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Use of uninitialized value $tlen_sum in division (/) at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
Illegal division by zero at /opt/chloroExtractor/bin/scale_reads.pl line 392, <R> line 10.
[18-02-13 16:06:10] [PipeWrap] scr exited:65280 
[18-02-13 16:06:10] [PipeWrap] HASH(0x3e41f10) at /opt/chloroExtractor/bin/../lib/PipeWrap.pm line 153.

--------------

*** DONE $seqwithtlen is first of all not initialized
*** DONE find bug and fix code 
when none TLEN is above 0, the if condition is never executed, so $seqwithtlen and $tlen_sum are both 0, by trying to calculate the insert size with 
int($tlen_sum/$seqwithtlen) you divide by 0 -> Crash

**** Code  (fixed)  
# my $c;
# my $tlen_sum; 
# my $seqwithtlen = 0;
#
#
# while(my $aln = $sp->next_aln()){
#    print BAM "$aln";
#    print FQ "@",$aln->qname,"\n", $aln->seq, "\n+\n",$aln->qual,"\n";
#    
#    # closest ref
#    my $id = $aln->rname();
#    my ($ref_id) = $id =~ /(NC_\d+)/;
#    $refs{$ref_id}++; # increment mapping on ref count
#
#    # insert size
#    if (abs($aln->tlen) > 0){
#        $tlen_sum+= abs($aln->tlen); # compute isize
#        $seqwithtlen++;
#    }
#
#    $c++;
#    last if $c >= $opt{max_reads}
# }       
#
#  print "coverage\t", $current_cov, "\n";
# print "genome_size\t", $genome_size, "\n";
# print "target_coverage\t", $opt{target_coverage}, "\n";
# if ($seqwithtlen > 0){ print "insert_size\t", int($tlen_sum/$seqwithtlen), "\n";}
# else {print "insert_size\tNA\n";}
# print "closest_ref\t", $closest_ref || 'NA', "\n";





     
* DONE Week 3 19.02 - 23.02
  - State "DONE"       from "PROGRESS"   [2018-03-01 Do 10:23]
  - State "PROGRESS"   from "TODO"       [2018-02-20 Di 10:14]

** Meeting
   https://github.com/chloroExtractorTeam/chloroplast_landscape/issues/9

** Table

|                 | Demo          | sim10         | sim100        | sim1000       | SRR1945435 | SRR1945436 | SRR1945437 | SRR1945438 | SRR1945439 | SRR1945440 | SRR1945441 | SRR1945442 | SRR1945443 | SRR1945444 | SRR1945445 |   |                                                       |
|                 |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   |                                                       |
|-----------------+---------------+---------------+---------------+---------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+---+-------------------------------------------------------|
| chloroExtractor | S             | F - coverage  | S             | S             | I          | I          | I          | I          | I          | I          | I          | I          | S          | I          | I          |   | ~25min ~16G RAM                                       |
|                 |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   |                                                       |
| NOVOPlasty      | S             | I - 3 contigs | I - 3 contigs | I - 3 contigs | E / I      | E / I      | E /I       | F / I      | E  / IS    | F / F      | E  / F     | E / IS     | E /I       | E /F       | F /F       |   | ~15min                                                |
|                 |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   | ~ 1h runtime ~50G RAM                                 |
| Fast-plast      |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   |                                                       |
| IOGA            | S - 3 contigs | F             | *R*           | *R*           | I          | I          | I          | I          | I          | I          | I          | *R*        | I          | I          | I          |   | ~ wow, runtime over 9000, so much RAM, very efficient |
|                 |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   |                                                       |
| GetOrganelle    | I             | E  / I        | E /I          | E / I         | E / I      | E / I      | E / I      | E / I      | E / I      | E / I      | E / I      | E / I      | E / I      | E / I      | E / I      |   | ~ 30min ~16G RAM                                      |
|                 |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   |                                                       |
| org.asm         | S             | S             | S             | S             | E / E      | E / I      | I / I      | F / F      | E  / I     | F  / F     | I  / I     | E / *R*    | I  / I     | I  / I     | F  / F     |   |                                                       |
|                 |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   |                                                       |
|                 |               |               |               |               |            |            |            |            |            |            |            |            |            |            |            |   |                                                       |

E = Error
F = Fail
I = Incomplete
S = Success
R = still running

** DONE Fix Error Problems
   - State "DONE"       from "PROGRESS"   [2018-02-21 Mi 09:30]
   - State "PROGRESS"   from "TODO"       [2018-02-20 Di 10:18]

*** DONE Chloroextractor

**** DONE divide by 0 bug

     Fixed and new Release 1.03 from 20.02.

     look at "Fixing chloroExtractor bug (scale reads) [week 2]

*** DONE GetOrganelle
    - State "DONE"       from "PROGRESS"   [2018-02-21 Mi 09:20]
    - State "PROGRESS"   from "TODO"       [2018-02-21 Mi 09:20]

**** err code 255
     == Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945438/SRR1945438_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: 255

     2018-02-13 16:08:21,913 - ERROR: Assembling failed.

     -> cant detect phred offset.

-> --spades-options "--careful --phred-offset 33"

**** err code -6

     2018-02-16 14:27:02,388 - ERROR: Error in SPAdes: 
     == Error ==  system call for: "['/opt/spades/bin/hammer', '/home/s229502/Master_thesis/runs/SRR1945438/GO/SRR1945438_1.fastq_GO_out/filtered_spades/corrected/configs/config.info']" finished abnormally, err code: -6


     2018-02-16 14:27:02,390 - ERROR: Assembling failed.

     -> Test more set -> same error on 3 other sets (38 / 40 / 43)

     -> try --assamble-only 

---> -w (Word size) / read size was over 100 we only have 100-readlength reads

*** DONE NOVOplasty

**** Invaild Seed
     -> BUG, fixed in version 2.6.4 from 20.Feb



** TODO Wrapper Skripts
   All skripts in /home/s229502/Master_thesis/skripts/

*** 0. Concept: The Blackbox
 (_1.fq)      |------------|
       \      | 1-->2--v   |  -> chloro.fa
ID /SRR ->    | v--4<--3   |  -> log_file
       /      | 5-->6----> |  -> benchmark_file
 (_2.fq)      |------------|  -> statisiks / evaluation


*** DONE 1. Directory structure
    - State "DONE"       from "PROGRESS"   [2018-02-22 Do 13:15]
    - State "PROGRESS"   from "TODO"       [2018-02-22 Do 11:06]

    SRR*******/  CE ----------------------> CE /  raw_data/ ---------------- raw data output dir from the prog
                 FP                               chloro.fa
		 GO                               log_file
		 IOGA                             benchmark_file
		 NP                               evaluation
		 O.G

   - Link runskripts / nessesary data / things like make_NP_conf.sh!
   (- create benchmark file)
--> change / improve cp_skript.sh!

*** DONE 2. run commands
    - State "DONE"       from "PROGRESS"   [2018-02-22 Do 13:49]
    - State "PROGRESS"   from "TODO"       [2018-02-22 Do 13:15]

    Change / improve run_*prog*.sh
    -input now SRR***** not SRR***_1.fastq / SRR***_2.fastq
    -Better_output
    
*** DONE 3. execute run.sh
    - State "DONE"       from "PROGRESS"   [2018-02-22 Do 14:04]
    - State "PROGRESS"   from "TODO"       [2018-02-22 Do 13:49]
    

    - make and start timer on benchmark_file
    change / improve sbatch_run_all

*** DONE 4. wait for programs to finish
    - State "DONE"       from "PROGRESS"   [2018-02-22 Do 15:26]
    - State "PROGRESS"   from "TODO"       [2018-02-22 Do 14:11]
    
    ????
    grep on squeue with JobID?
    
    end timer for benchmark_file
    
*** DONE 5. Find importent files    
    - State "DONE"       from "PROGRESS"   [2018-02-27 Di 08:43]
    - State "PROGRESS"   from "TODO"       [2018-02-22 Do 14:11]

    CE: -log = slurm
        -fa  = *Out-dir*/fcg.fa

    FP: -log = *out*_Fast-Plast_Progress.log
               *out*_results_error.log
               *out*_results.out.log
        -fa  = *out-dir*/Final_Assembly/*out*_FULLCP.fsa || *out*_afin_iter*.fa

    GO: -log = slurm || *out*/get_org.log.txt
        -fa  = *out*/filtered_spades/contigs.fasta

  IOGA: -log = slurm
        -fa  = $VAR="$(grep 'highest ALE score' slurm* | grep -oP 'SRR.*contigs')"
               *out*/$VAR.fasta

    NP: -log = log_*out*.txt
        -fa  = Option*_*out*.fasta   -> replace all * with " sed -r 's/\*/N/g' > chloro.fa"

   O.G: -log = slurm
        -fa  = *.fa


copy them to folder

*** DONE 6. Evaluate
    - State "DONE"       from "PROGRESS"   [2018-02-27 Di 18:01]
    - State "PROGRESS"   from "TODO"       [2018-02-27 Di 08:43]
    
    - Report success / Fail / ERROR
    - SeqFilter ( bp count / amount contigs )
    - AliTV (need ref Chloroplast genom)

    ls */NOVOPlasty/NP_*.log | rev | cut -f1 -d_ | rev |cut -f1 -d. | sort | uniq -c
**** Success
     154Kb -+ 10% 
     1 Seq
**** Partial /   contigs
     154 -+ 10% (bzw - 1 IR )
     contigs egal
     
**** Incomplete - low
     contigs egal
     < 154 - 1IR -+ 10%
**** Incomplete - high      
     > 
**** ERROR / FAIL
     NO FASTA

**** TABLE
|     | CE | FP |
|     |    |    |
|-----+----+----|
| SRR |    |    |
| SRR |    |    |
|     |    |    |
#+BEGIN_SRC sh :dir /home/s229502/Master_thesis/Wrapper/test_auto/
bash ev_stat.sh
#+END_SRC

#+RESULTS:
| SRR         | CE         | FP             |  GO             | IOGA       | NP      | OA             |
| SRR1945446: | ERROR_file | PARTIAL        | INCOMPLETE_high | PARTIAL    | PARTIAL | PARTIAL        |
| SRR1945451: | SUCCESS    | SUCCESS        | INCOMPLETE_high | PARTIAL    | ERROR   | ERROR          |
| SRR1945453: | PARTIAL    | PARTIAL        | INCOMPLETE_high | PARTIAL    | ERROR   | ERROR          |
| SRR1945454: | SUCCESS    | SUCCESS        | INCOMPLETE_high | PARTIAL    | ERROR   | ERROR          |
| SRR1945455: | PARTIAL    | INCOMPLETE_low | PARTIAL         | PARTIAL    | ERROR   | PARTIAL        |
|             |            |                |                 |            |         |                |

# | TEST:       | ERROR_file | ERROR_file | ERROR_file      | ERROR_file | ERROR_file | ERROR_file     |
# NO Paired reads
# | SRR1945447: | ERROR_file | ERROR_file | ERROR_file      | ERROR_file | ERROR_file | ERROR_file     |
# | SRR1945448: | ERROR_file | ERROR_file | ERROR_file      | ERROR_file | ERROR_file | ERROR_file     |
# | SRR1945449: | ERROR_file | ERROR_file | ERROR_file      | ERROR_file | ERROR_file | ERROR_file     |
# | SRR1945450: | ERROR_file | ERROR_file | ERROR_file      | ERROR_file | ERROR_file | ERROR_file     |
# | SRR1945452: | ERROR_file | ERROR_file | ERROR_file      | ERROR_file | ERROR_file | ERROR_file     


* DONE Week 4 26.02 - 02.03
  - State "DONE"       from "PROGRESS"   [2018-03-13 Di 09:05]
  - State "PROGRESS"   from "TODO"       [2018-03-01 Do 10:23]
** CANCELED 26.02 - *Ill on 26.02*
** Meeting 28.02
*** Look into Novoplasty /write issue to devteam
*** benchmark-tests : time
    for each singularity one time - sum up
    time in singularity
    
    - docker stats? singularity stats?
*** chloroExtractor
    - uni-cycler still in dev
    - coverage? 200x  10% down each time
    - 
     
*** Goal: start all 6 on 1001
*** 300er reads (provided by markus)
    
** start 42 Data 
*** ev all 

* CANCELED Week 5 05.03 - 09.03
  - State "CANCELED"   from ""           [2018-03-16 Fr 11:32]
  *Retreat and ILL!*

* DONE Week 6 12.03 - 16.03
  - State "DONE"       from "PROGRESS"   [2018-03-26 Mo 10:20]
  - State "PROGRESS"   from "TODO"       [2018-03-12 Mo 10:32]
** DONE Rerun Chloro with re-run-cov feature
*** DONE Build rerun in script
** Mapping Data on ref to controll cov analysis
  

* DONE Week 7 19.03 - 23.03
  Run all 6 on the 57 sets from the GO preprint

  - Automating evalutation difficult... some fastg Graphes could be finished by hand.

* DONE Week 8 26.03 - 29.03 (30.03 Karfreitag)
  Still several programs are running on the 57 form the GO preprint

** Meeting

    Contact:
    GetOrganelle, Parameters for each SRR?
    IOGA, runtime Issues

    Try to formate Data for NOVOPlasty -> header Issue

    AliTV for evaluating NOVOPlasty successes

    start 1001 genom on CE and FP

** Table
#+BEGIN_SRC sh :dir ~/Master_thesis/runs/prepint_GO_5M
bash ev_stat.sh|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE             | FP             | GO             | IOGA           | NP             | OA             |
|-------------+----------------+----------------+----------------+----------------+----------------+----------------|
| ERR2144882: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | ERROR          | ERROR          |
| ERR268390:  | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | SUCCESS        | SUCCESS        |
| ERR268421:  | INCOMPLETE_low | PARTIAL        | INCOMPLETE_low | INCOMPLETE_low | PARTIAL        | SUCCESS        |
| ERR845262:  | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     |
| ERR964904:  | ERROR          | INCOMPLETE_low | INCOMPLETE_low | ERROR          | Running        | ERROR          |
| SRR072296:  | PARTIAL        | PARTIAL        | INCOMPLETE_low | PARTIAL        | SUCCESS        | ERROR          |
| SRR1145775: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR1919543: | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     |
| SRR1925329: | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | PARTIAL        | Running        | PARTIAL        |
| SRR1946759: | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     | ERROR_file     |
| SRR3089810: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | PARTIAL        | SUCCESS        |
| SRR3656962: | SUCCESS        | SUCCESS        | Running        | PARTIAL        | PARTIAL        | INCOMPLETE_low |
| SRR4099903: | PARTIAL        | SUCCESS        | PARTIAL        | ERROR          | Running        | ERROR          |
| SRR4099904: | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | ERROR          |
| SRR4428742: | PARTIAL        | SUCCESS        | PARTIAL        | ERROR          | SUCCESS        | PARTIAL        |
| SRR5028798: | SUCCESS        | SUCCESS        | PARTIAL        | PARTIAL        | Running        | PARTIAL        |
| SRR5152976: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602572: | SUCCESS        | PARTIAL        | Running        | ERROR          | Running        | ERROR          |
| SRR5602573: | PARTIAL        | PARTIAL        | Running        | ERROR          | Running        | ERROR          |
| SRR5602574: | ERROR          | SUCCESS        | Running        | ERROR          | Running        | ERROR          |
| SRR5602575: | INCOMPLETE_low | SUCCESS        | Running        | ERROR          | Running        | ERROR          |
| SRR5602576: | ERROR          | PARTIAL        | SUCCESS        | ERROR          | Running        | INCOMPLETE_low |
| SRR5602577: | PARTIAL        | PARTIAL        | Running        | ERROR          | Running        | ERROR          |
| SRR5602578: | INCOMPLETE_low | SUCCESS        | PARTIAL        | ERROR          | Running        | ERROR          |
| SRR5602579: | ERROR          | PARTIAL        | Running        | ERROR          | Running        | ERROR          |
| SRR5602581: | INCOMPLETE_low | SUCCESS        | PARTIAL        | ERROR          | Running        | ERROR          |
| SRR5602582: | SUCCESS        | SUCCESS        | PARTIAL        | PARTIAL        | Running        | ERROR          |
| SRR5602583: | ERROR          | INCOMPLETE_low | PARTIAL        | INCOMPLETE_low | Running        | ERROR          |
| SRR5602584: | SUCCESS        | SUCCESS        | PARTIAL        | ERROR          | Running        | SUCCESS        |
| SRR5602585: | SUCCESS        | PARTIAL        | PARTIAL        | PARTIAL        | Running        | ERROR          |
| SRR5602586: | INCOMPLETE_low | PARTIAL        | SUCCESS        | ERROR          | Running        | PARTIAL        |
| SRR5602587: | ERROR          | PARTIAL        | PARTIAL        | ERROR          | Running        | ERROR          |
| SRR5602588: | SUCCESS        | PARTIAL        | Running        | PARTIAL        | Running        | SUCCESS        |
| SRR5602589: | SUCCESS        | SUCCESS        | Running        | ERROR          | Running        | SUCCESS        |
| SRR5602590: | PARTIAL        | PARTIAL        | Running        | PARTIAL        | Running        | ERROR          |
| SRR5602591: | INCOMPLETE_low | SUCCESS        | PARTIAL        | PARTIAL        | Running        | ERROR          |
| SRR5602592: | SUCCESS        | SUCCESS        | Running        | PARTIAL        | Running        | ERROR          |
| SRR5602593: | SUCCESS        | SUCCESS        | PARTIAL        | PARTIAL        | Running        | ERROR          |
| SRR5602594: | INCOMPLETE_low | SUCCESS        | Running        | PARTIAL        | Running        | ERROR          |
| SRR5602595: | ERROR          | INCOMPLETE_low | PARTIAL        | ERROR          | Running        | Running        |
| SRR5602596: | INCOMPLETE_low | PARTIAL        | Running        | PARTIAL        | Running        | ERROR          |
| SRR5602597: | PARTIAL        | SUCCESS        | Running        | ERROR          | Running        | ERROR          |
| SRR5602598: | ERROR          | SUCCESS        | Running        | PARTIAL        | Running        | INCOMPLETE_low |
| SRR5602599: | SUCCESS        | SUCCESS        | Running        | PARTIAL        | Running        | SUCCESS        |
| SRR5602600: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | Running        | ERROR          |
| SRR5602601: | PARTIAL        | PARTIAL        | PARTIAL        | ERROR          | Running        | ERROR          |
| SRR5602602: | SUCCESS        | SUCCESS        | Running        | PARTIAL        | Running        | SUCCESS        |
| SRR5602603: | ERROR          | SUCCESS        | Running        | PARTIAL        | Running        | ERROR          |
| SRR5602604: | PARTIAL        | SUCCESS        | Running        | PARTIAL        | Running        | ERROR          |
| SRR5602605: | SUCCESS        | PARTIAL        | PARTIAL        | ERROR          | Running        | ERROR          |
| SRR5602606: | INCOMPLETE_low | SUCCESS        | Running        | PARTIAL        | Running        | SUCCESS        |
| SRR5602607: | INCOMPLETE_low | PARTIAL        | PARTIAL        | PARTIAL        | Running        | ERROR          |
| SRR5602608: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | Running        | ERROR          |
| SRR5602609: | ERROR          | SUCCESS        | Running        | PARTIAL        | Running        | ERROR          |
| SRR5602610: | ERROR          | PARTIAL        | PARTIAL        | INCOMPLETE_low | Running        | ERROR          |
| SRR954512:  | PARTIAL        | PARTIAL        | PARTIAL        | ERROR          | Running        | SUCCESS        |
| SRR955759:  | SUCCESS        | SUCCESS        | PARTIAL        | ERROR          | Running        | SUCCESS        |


* DONE Week 9 03.04 - 06.04 (02.04. Ostermontag)

  restart several runs from CE and FP, memory issues and singularity fails 

  AliTV on them to compare

  start GO on those which cant finished by CE and FP, 



** AliTV

#+BEGIN_SRC sh :dir ~/Master_thesis/runs/Athaliana
cat list_AliTV |column -t
#+END_SRC

#+RESULTS:
| SRR         | CE             | FP              | GO      | IOGA | NP | OA |
| SRR1945451: | SUCCESS        | SUCCESS         | -       | -    | -  | -  |
| SRR1945445: | INCOMPLETE_low | INCOMPLETE_low  | Running | -    | -  | -  |
| SRR1945454: | SUCCESS        | SUCCESS         | -       | -    | -  | -  |
| SRR1946223: | SUCCESS        | INCOMPLETE_high | Running | -    | -  | -  |
| SRR1946425: | SUCCESS        | INCOMPLETE_high | Running | -    | -  | -  |
| SRR1946405: | SUCCESS        | INCOMPLETE_low  | Running | -    | -  | -  |
| SRR1945791: | INCOMPLETE_low | INCOMPLETE_low  | Running | -    | -  | -  |
| SRR1946553: | SUCCESS        | SUCCESS         | -       | -    | -  | -  |
| SRR1946316: | INCOMPLETE_low | SUCCESS         | -       | -    | -  | -  |
| SRR1945624: | ERROR          | PARTIAL         | Running | -    | -  | -  |
| SRR1945998: | ERROR          | INCOMPLETE_low  | Running | -    | -  | -  |

** Stats
#+BEGIN_SRC sh :dir ~/Master_thesis/runs/Athaliana                                                                                                                                                                 
bash percent_stat.sh stat_AT_09-04.log |column -t                                                                                                                                                                                          
#+END_SRC 

#+RESULTS:
| Tool    | SUCCESS | %    | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      |     119 | ~16% |    45 |       2 |     596 |     184 |       0 |       |
| FP      |     222 | ~29% |    28 |       8 |     504 |     184 |       0 |       |
| GO      |       0 | ~0%  |   137 |       0 |     403 |       0 |       0 |       |
| IOGA    |       0 | ~0%  |     0 |       0 |       0 |       0 |       0 |       |
| NP      |       0 | ~0%  |     0 |       0 |       0 |       0 |       0 |       |
| OA      |       0 | ~0%  |     0 |       0 |       0 |       0 |       0 |       |
| Summary |     255 | ~33% |     - |       - |       - |     184 |       - |   946 |

#+BEGIN_SRC sh :dir ~/Master_thesis/runs/Athaliana                                                                                                                                                                 
bash percent_stat.sh stat_AT-25-04.log |column -t                                                                                                                                                                                          
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS | %    | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      |     136 | ~15% |    54 |       3 |     706 |      47 |       0 |       |
| FP      |     266 | ~30% |    29 |      11 |     593 |      47 |       0 |       |
| GO      |       0 | ~0%  |   137 |       0 |     403 |       0 |       0 |       |
| IOGA    |       0 | ~0%  |     0 |       0 |       0 |       0 |       0 |       |
| NP      |       0 | ~0%  |     0 |       0 |       0 |       0 |       0 |       |
| OA      |       0 | ~0%  |     0 |       0 |       0 |       0 |       0 |       |
| Summary |     303 | ~34% |     - |       - |       - |      47 |       - |   946 |

* Week 10 09.04 - 13.04
** PROGRESS Variance calling
   - State "PROGRESS"   from "TODO"       [2018-04-10 Di 10:15]
*** Between the Programs
*** Within the Programs
    - Start every Success, which both had ( 86 ) , 10 more times. To see how they behave
**** Normal
***** list
#+BEGIN_SRC sh :dir ~/Master_thesis/runs/Athaliana
cat stat_AT_09-04.log | grep -oP "SRR\d+.*SUCC.*SUCC.*"|column -t

#+END_SRC

#+RESULTS:
| SRR1945451: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945454: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945473: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945486: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945487: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945490: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945494: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945520: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945527: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945531: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945534: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945538: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945548: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945549: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945555: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945562: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945569: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945582: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945603: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945758: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945764: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945773: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945783: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945796: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945817: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945831: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945850: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945864: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945898: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945940: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945990: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945991: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1945999: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946010: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946053: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946054: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946105: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946136: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946153: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946188: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946209: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946219: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946233: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946258: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946260: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946268: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946270: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946278: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946288: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946305: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946312: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946313: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946386: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946389: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946391: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946399: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946401: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946404: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946409: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946410: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946415: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946426: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946433: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946436: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946437: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946440: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946441: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946442: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946443: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946447: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946452: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946453: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946456: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946457: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946463: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946468: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946471: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946478: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946479: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946480: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946481: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946553: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946556: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946557: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946560: | SUCCESS | SUCCESS | - | - | - | - |
| SRR1946562: | SUCCESS | SUCCESS | - | - | - | - |
***** Table - Percent
#+BEGIN_SRC sh :dir ~/Master_thesis/runs/Athaliana
bash percent_stat.sh log_both
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V1
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_01/
cat stat_var01|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
|             |         |                 |    |      |    |    |

***** Table - percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_01/
bash percent_stat.sh stat_var01
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 77 ~90%   |     0 |       0 |       9 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V2 
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_02/
cat stat_var02|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
***** Table - percent 
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_02/
bash percent_stat.sh stat_var02
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 78 ~91%   |     0 |       0 |       8 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V3
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_03/
cat stat_var03|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | PARTIAL         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |

***** Table - Percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_03/
bash percent_stat.sh stat_var03
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 76 ~88%   |     0 |       1 |       9 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V4
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_04/
cat stat_var04|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
***** Table percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_04/
bash percent_stat.sh stat_var04
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 79 ~92%   |     0 |       0 |       7 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V5
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_05/
cat stat_var05|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |

***** Table - percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_05/
bash percent_stat.sh stat_var05
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 78 ~91%   |     0 |       0 |       8 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |
**** V6
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_06/
cat stat_var06|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | PARTIAL         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | ERROR           | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |

***** Table - percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_06/
bash percent_stat.sh stat_var06
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 69 ~80%   |     9 |       1 |       7 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V7
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_07/
cat stat_var07|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | PARTIAL         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
***** Table - percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_07/
bash percent_stat.sh stat_var07
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 79 ~92%   |     0 |       1 |       6 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V8
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_08/
cat stat_var08|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
***** Table - percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_08/
bash percent_stat.sh stat_var08
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 79 ~92%   |     0 |       0 |       7 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V9 
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_09/
cat stat_var09|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP              | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | INCOMPLETE_high | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | INCOMPLETE_low  | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS         | -  | -    | -  | -  |
***** Table - percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_09/
bash percent_stat.sh stat_var09
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 75 ~87%   |     0 |       0 |      11 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

**** V10
***** list
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_10/
cat stat_var10|column -t
#+END_SRC

#+RESULTS:
| SRR         | CE      | FP             | GO | IOGA | NP | OA |
| SRR1945451: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945454: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945473: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945486: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945487: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945490: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945494: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945520: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945527: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945531: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945534: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945538: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945548: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945549: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945555: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945562: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945569: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945582: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945603: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945758: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945764: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945773: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945783: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945796: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945817: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945831: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945850: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945864: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945898: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945940: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945990: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945991: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1945999: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946010: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946053: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946054: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946105: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946136: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946153: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946188: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946209: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946219: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946233: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946258: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946260: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946268: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946270: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946278: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946288: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946305: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946312: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946313: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946386: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946389: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946391: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946399: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946401: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946404: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946409: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946410: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946415: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946426: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946433: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946436: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946437: | SUCCESS | INCOMPLETE_low | -  | -    | -  | -  |
| SRR1946440: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946441: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946442: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946443: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946447: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946452: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946453: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946456: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946457: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946463: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946468: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946471: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946478: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946479: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946480: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946481: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946553: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946556: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946557: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946560: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
| SRR1946562: | SUCCESS | SUCCESS        | -  | -    | -  | -  |
***** Table - percent
#+BEGIN_SRC sh :dir  ~/Master_thesis/runs/Variance_AT_10/
bash percent_stat.sh stat_var10
#+END_SRC

#+RESULTS:
| Tool    | SUCCESS % | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      | 86 ~100%  |     0 |       0 |       0 |       0 |       0 |       |
| FP      | 78 ~91%   |     0 |       0 |       8 |       0 |       0 |       |
| GO      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| IOGA    | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| NP      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| OA      | 0 ~0%     |     0 |       0 |       0 |       0 |       0 |       |
| Summary | 86 ~100%  |     - |       - |       - |       0 |       - |    86 |

** DONE Preprint-Go: start witch adjusted Parameters from Jing Jang-Jiung
   - State "DONE"       from "PROGRESS"   [2018-04-17 Di 14:59]
   - State "PROGRESS"   from "TODO"       [2018-04-13 Fr 11:12]
   - Over the Weekend
   - Not really more ....

* Week 11 16.04 - 20.04 
** Variance calling
*** PROGRESS Aligments
    - State "PROGRESS"   from "TODO"       [2018-04-17 Di 15:01]






** Problems wioth gaias....
* Week 12 23.04 - 27.04
** Still Variance calling
* Week 13 30.04 - 04.05
** Bandage is a shitty Tool
** SPAdes overlaps?!
* CANCELED Week 14 07.05 - 11.05 (ILL)
  - State "CANCELED"   from ""           [2018-05-14 Mo 08:58]
* Week 15 14.05 - 18.05 
** Variance Calling
*** BWA mapping
*** MiniMap Mapping
*** Detect IR script
*** SNPs Tab
*** FCG.pl overlap SPAdes
Bandage source code?
diffrent kmer sized overlaps...
-> Frank refinment fcg

*** Structure Variations - easy or problem?
Structure varianten etabliren (zur not mit der hand) und mappen um zu erkennen ob diese probleme machen? (1000 bp?)

*** Convention reihenfolge?
Fast-plast uses:

Hi Simon,
​
LSC and SSC identity are assigned based on relative size of the SC regions.
​
Orientation is determined by looking at the relative orientation of the rpl and rps genes in the LSC, all genes in the SSC, and the rrn rRNAs in the IR. The code orientates the LSC so there are more "-" 
strand rps and rpl genes than "+", more "-" strand than "+" strand genes in the SSC, and with rrn genes on the "-" strand for the IRA. This works for most lineages (that we know of) in angiosperms. 
In reality, the SSC is probably in both directions across copies of the plastome in a plant. This is more for convention than anything else.
​
Things look like this:
​
<-------------LSC-------------><-------IRB-------><--------SSC--------><-------IRA------->
<psbA(-)--------------rpl22(-)><------rrn23(+)---><ndhF(-)---ndhD(-)--><------rrn23(-)--->
​
This method doesn't work with plastomes that deviate from this common orientation. If there are major rearrangements, I do more of this step-wise using the scripts found in the bin directory of the Fast-Plast repository.
​
--Michael

 
* Week 16 22.05 - 25.05 (21.05 Pfingstmontag)
** bcftools / vcftools
Variant calling (samtools things) 
writing scripts...
finally everything works, now waiting for fcg-fix...

* Week 17 28.05 - 01.06
** Struckturelle änderungen
Not visable in the VC files, maybe CIGAR Strings?
Visable in those but none idea how to evaluate or find...

** Vergleichen SNPs indels mit 1001 Genom project
there are diffrent SNPs,
Tair10 right ref?

** Verschiedene Varianten vorhanden?
...maybe later

* Week 18 04.06 - 08.06
** Remapping - real SNPs?
Just some diffrences at the start of the chloroplast - mapping problem?

Assambler verschchiedene Varianten?
wenn verschiedene Reads vorkommen -> homozygot?

** Structure variations?
Still no idea how to detect them, if there are some...
CIGAR STRINGS: skript that evaluates all Cigar strings, and matches them to find pairs and maybe a inversion? #showerthoughts
Programs to check: DELLY, breakdance, NMOPS, Control-FREEC
*** DELLY
Docker image available

*** breakdancer
bam2cfg.pl doesn't work....
No Error massage at all...

* Week 19 11.06 - 15.06 (Markus vacations)
** Call with Frank monday or tuesday or wednesday....
   - Puplikation lesen 1001 Genom -Arthur
   - Phänotype, gwas. Jan Fragen.
      - Korrelieren von "besonderheiten" von SNPS SV Indels, vorhersagen von Phänotyp
      - Phänotypen in Chloroplasten sichtbar?
      - Welche Daten benötigt?
   - Remapping auf allen zum SNPs
   - Neuen fastg parser testen! - still not all overlaps are removed... :( 
   - 
** Niklas Drosera Data durchlaufen lassen 

   130314_I269_FCC1KFEACXX_L4_SZAXPI024351-16_1.fq.gz.clean.dup.clean bzw 
   130314_I269_FCC1KFEACXX_L4_SZAXPI024351-16_2.fq.gz.clean.dup.clean
   in /storage/compevolbiol/projects/carnivores/data/Drosera/Illumina/illumina_raw
** Server auslasen LOL
NCBI search term:

((((((("green plants"[orgn]) AND "wgs"[Strategy]) AND "illumina"[Platform]) AND "biomol dna"[Properties]) AND "paired"[Layout]) AND "random"[Selection])) AND "public"[Access] 

-> download SRA list
-------

perl -F"," -ane 'print if $F[6]>399 and $F[3]>999999' SraRunInfo_plants.csv | grep -vf species_cpbase.list | sort -u -t, -k29,29 | shuf


-----

#!/bin/bash
​
curl http://rocaplab.ocean.washington.edu/old_website/tools/cpbase/run | grep href | grep genome | grep td | cut -f3 -d ">" | cut -f1 -d"<" 

* Week 20 18.06 - 22.06 (Markus vacations)

* Less Importend stuff

** DONE Fix Docker error for perl
   - State "DONE"       from "PROGRESS"   [2018-04-03 Di 11:15]
   - State "PROGRESS"   from "TODO"       [2018-04-03 Di 11:15]

   perl: warning: Setting locale failed.
   perl: warning: Please check that your locale settings:
        LANGUAGE = (unset),
        LC_ALL = (unset),
        LANG = "de_DE.UTF-8"
    are supported and installed on your system.
    perl: warning: Falling back to the standard locale ("C").

    --> Needs sudo rights to "sudo dpkg-reconfigure locales 
                              sudo locale-gen de_DE.UTF-8"


			      
  FIX: in singularity exec:       bash -c "export LC_ALL="C.UTF-8" && <command>"
 

** TODO All Docker in one Repo

** DONE Error message for chloroExtractor, when no peak is found - scaling reads! Error deivision by 0
   - State "DONE"       from "CANCELED"   [2018-02-15 Do 16:00]
   - State "CANCELED"   from ""           [2018-02-15 Do 10:08]
     moved ( not less importend - VERY IMPORTEND)

   DONE!


** TODO IOGA fork and fix 
   https://github.com/holmrenser/IOGA/blob/master/IOGA.py#L334
   

** DONE Lin Tao - Problem, mitochondria data in chloroExtractor
   - State "DONE"       from "PROGRESS"   
   - State "PROGRESS"   from "TODO"      
*** DONE Download all Data
*** DONE Run all Data on CE
*** blast to verify Mito seqs
    yeah, the blast showed mitochondrial parts in some fraquments (>2,5 kb)
    
    Thoughts: It seems it is some part of rbcL large subunit - which could be in mitochondria....
   
* Importend executables / shortcuts 
** exe 
  
  ~/singularity/bin/singularity shell -w docker://ubuntu
  which doenst work anymore with 2.4 *yeah* 

** shortcuts
   alt+F2 Anwendungsfilter... merks dir endlich ma -.-
** orgmode code
#+BEGIN_SRC sh :dir 

#+END_SRC


* Summary
** Title
Seperating the good from the bad...
Exploring the genomic landscape of chloroplasts from genomic sequencing data
** docker container for all programs (IOGA, NOVOplasty, GetOrganelle, OrgAsm, chloroExtractor, fast-plast)
- We wanted an automatic version of all programs to be as reproducable as possible
** What is "the best" program ( Time / efficency / Handling / fully automatic )
- best one fast-plast and chloroExtractor ->
  then used on all data sets
** Data used
*** Simulated Data (A.thaliana)
*** Athaliana 1001 Genom 
**** 303 / 1135
*** GO preprint
*** try to find new ones (nocpbase)
**** 17 / 49
*** Drosera
*** TAIR10 as reference
** Variance Calling ( CE / FP - Assambler influence? )
*** CE consistent no influances at all
*** FP not reproducable 
** Structure Variance? ( CE / FP )
*** TODO
** Remapping / Backmapping ( Hetero / Homoplasty - Real SNPS? )
*** no Heteroplasty
** GWAS
*** TODO
** Programs used
*** Main
**** chloroExtractor*
**** fast-plast*
**** IOGA
**** NOVOPlasty
**** GetOrganelle
**** OrgASM
*** Second
**** wrapper skripts (bash)
**** bowtie2
**** minimap2
**** samtools
**** Docker / singularity
*** Evaluation
**** Bandage
**** AliTV
**** several skripts (bash / perl)
**** R (GWAS)
**** Delly
**** vcf / bcftools
** Bugfixes
*** mostly CE bugfixes

* Story / intro
** Title
Seperating the good from the bad...
Exploring the genomic landscape of chloroplasts from genomic sequencing datatreduction
** Open Science? / Reproducable science?
** Chloroplasts are cool!
*** Because is so!
**** very conserved in structure and and genome
Chloroplasts are very importent for plants, they are make photosynthese. Because of this importense there is no much room for failure, this is one reason why chloroplasts are very conserved in genomic and structure.
**** Pic chloroplast LSC - IRB - SSC - IRA / Bandage 
A chloroplast genome contains, two single copies, the large and the small one *~size of each SC*, which are linked by two inverted repeats. *CHLOROG GENOM PIC* *BANDAGE PIC?* 

/Chloroplast genomes of plants are known to be highly conserved in both gene order and gene content (Raubeson and Jansen 2005)./
/They exhibit a substitution rate much lower than nuclear DNA, which is even significantly reduced in the inverted repeat regions (Wolfe et al. 1987)./


**** some chloros without IRB
**** differences/range in sizes 
*** Very important for marker and phylogeny in Plants
**** rbc / rubisco
**** Maybe the whole chloroplast as marker, not just several genes

/Recently comparative analyses of complete chloroplast sequences showed that the perception of low variation of chloroplasts within species is wrong when looking at the genomic scale/
/(Whittall et al., 2010; Besnard et al., 2011; Kane et al., 2012). Kane et al. (2012) suggested that the whole chloroplast genome could be used as an ultra-barcode for identifying plant varieties./ 
/Furthermore, using one or few regions of the chloroplast genome is not the appropriate approach to describe the level of variability of the chloroplast genome./

** Data allready there, but not used many times ( Data in whole genome data ) / BIG DATA era
*** 1. BIG DATA / High Throuput -> Automatisation -> Bioinformatics?
We live in a time, in which you can get good amounts of genomic data for low money. This is possible because of better sequencing technologys in past past view years like Next Generation Sequencing and 3rd Generation Sequencing. 
Mostly Illumina or Pacbio and the with this technology corresponding highthrougput. *LOOK UP PRIZES* *ILLUMINA VS. PAKBIO?*
This is often refert as big data or big data era in a biological sense. In Biology there is overwhelmingly much data, espacialy genomic data, that some kind of automatisation in evaluation of data is needed. These automatisation
can only be achieved with knowledge in both biology and informatics. So the request in bioinformaticians and their tools has grown a large in the past years.  
*** 2. Data in genomic data -> nucleus / core extraction expensive and many more chloros then core
Since open sience is also grown with the flood of data, most of this data and also the tools to process and evaluate it, is open for everyone* *LICENSES?*. Because of this much data, the answer to new biological questions may be
anwseralbe just by using existing data. In genomic plant data for example. For plant squencing ofentimes tissues were used, which contains a lot of chloroplasts (e.g. Leafs). For the genomic questions chloroplasts are rather
uninteresting* *REALLY?*, beacause they have their own genome. But a nucleus or cellcore extraction can be expensive so it is not used much. So lot of genomic plant data also contains chloroplast or mitochondrial data.  
*** 3. Data in Data -> chloroExtractor
To use this chloroplast data in genomic plant data we (github.com/chloroExtractorTeam) wrote a programm called chloroExtractor *CITATION ChloroExtractor*, which assambles chloroplasts deNovo from those datasets. So to anwser question on the chloroplast landscape
across diffrent plants, there is no need for me to produce new raw data, because there is enough open source data which i can use. 
*** Squencing (?) Illumina vs. PacBio (?) -> better SV calling?
** Differences in species ~chloroplast landscape -> GWAS (phenotypes / SNPS )
*** GWAS
GWAS is short for Genome-wide association study *ARTHUR GWAS PAPER*. It trys to associate traits like flowering time in Arabidopsis thaliana with single nucleotide polymorphism (SNPs).  
*** 1001 Genom project on whole genom, maybe also effects can also be seen only on chloros
The 1001 Genom project, which squenced 1135 different Arabidosis thaliana *1001 GENOM PAPER* focused their GWAS on the whole genome. But maybe the chloroplast itself also contains SNPs which associate with certain 
traits. The traits were used from *ARAPHENO*. 
* Material / Methodes
** Infrastructure used (?)
** Introducing each tool
I used different tools to accomblish the extraction of chloroplast DNA from whole genome data. For each tool there was build a docker container to ensure reproducabilaty.  

pipeline
which assambler?
need ref?
coding language
automized?
*** chloroExtractor
The chloroExtractor *REF CHLORO* is the tool which was programmed by the chloroExtractorteam *GITHUB LINK*  from the CCTB *LINK*. ChloroExtractor uses a kmer based and BLAST+ approach to detect the chloroplast
genom in the data. With this approach the choloroExtractor is able to assamble the chloroplast DE NOVO, this means no reference chloroplast is needed. In order to get the kmers chloroExtractor uses
Jellyfish *JELLYFISH LINK*. Kmers of a special region *KMER PLOT* are blasted to verifie that these are chloroplast reads.  To assamble the kmers SPAdes is used. To detect the correct chloroplast the De Brujin graph-based
method form SPAdes is used. The main parts of the tool are programmed in perl. 

*** fast-plast
Fast-plast *GITHUBLINK* is another program to extract and find chloroplast genome sequences and mainly programmed in perl and C++. It uses also SPAdes but also Bowtie1, Bowtie2 *LINK BOWTIE* for mapping and also BLAST+ for 
gene conntent checking. With this approach fast-plast assambles chloroplasts also DE NOVO. 

*** Org.asm
The ORGanelle ASeMbler (ORG.asm *LINK*) is mostly wirten in phyton and trys to assamble over-represented small sequences in shotgun squence datasets, these could be chloroplasts. 

*** GetOrganelle
GetOrganelle *GETORGANELLE PAPER* is the first program in this list, which uses a reference, but only to detect the reads using Bowtie2 and BLAST+. It still de novo assambles the contigs using SPAdes.  
The last step of this one can not be automated. You have to open Bandange *BANDAGE LINK* to visualize the fastg-graph produced by SPAdes and put the chloroplast together per hand.

*** NOVOPlasty
NOVOPlasty *PAPER LINK* is only wirtten in PERL and has no dependencis at all. The NOVOPlasty uses one configuration file to handle all inputs, from inputfiles to options.   
 
** Comparison different tools to extract Chloroplasts out of genomic data
To evaluate which of the programs perfom best, I compared the tools on different subjects. First, the program handling, so how easy to use is the tool and how well it is documentated.
My goal was to automize all the tools, so i created for docker container for all of them which had none before *CHLOROEXTRACTORTEAM DOCKER LINKS*. Exept ORG.asm every tool was fairly
easy to install and use, but the ORG.asm i was only able to use in a docker container at all. It also the only tool which uses more than one command to start the pipline for extracting.
To automize NOVOPlasty which gets all its options and inputs from a single configuration file, i created a skript to build this.
For testing i used diffrent Data sets. First i used simulated Data with different chloroplast genome to plant genom ratios (GENOME : CHLOROPLAST: 1:10 / 1:100 / 1:1000). *Link Material - Method SimDATA*. 
For further tests I used different SRAs form the 1001 Genom Project (*SRA NUMBERS / Athaliana*). 
| Sim(Genome:Chloroplast)                       | CE  | FP | NP  | GO | OA | IOGA |
| SRA (1001 Genom / Athaliana                   |     |    |     |    |    |      |
|                                               |     |    |     |    |    |      |
|-----------------------------------------------+-----+----+-----+----+----+------|
| 1:10                                          | S   | S  | S-3 | E  | S  | E    |
| 1:100                                         | S   | S  | S-3 | E  | S  | -    |
| 1:1000                                        | S   | S  | S-3 | E  | S  | -    |
| SRR1945435                                    | I-5 | I  | I-4 | *  | E  | I-6  |
| SRR1945436                                    | I-6 | S  | I-3 | *  | I  | I-8  |
| SRR1945437                                    | I-5 | I  | I-4 | *  | I  | I-10 |
| SRR1945438                                    | S-3 | S  | I-6 | *  | E  | I-10 |
| SRR1945439                                    | I-4 | S  | I-1 | *  | I  | I-10 |
| SRR1945440                                    | I-4 | S  | E   | *  | E  | I-9  |
| SRR1945441                                    | I-5 | S  | E   | *  | I  | I-6  |
| SRR1945442                                    | I-4 | I  | I-1 | *  | -  | -    |
| SRR1945443                                    | S   | I  | I-2 | *  | I  | I-8  |
| SRR1945444                                    | I-4 | I  | E   | *  | I  | I-8  |
| SRR1945445                                    | I-4 | I  | E   | *  | E  | I_7  |
|                                               |     |    |     |    |    |      |
| * GetOrganelle finished but could             |     |    |     |    |    |      |
|   not easily finished in Bandage per hand     |     |    |     |    |    |      |

| SRR         | CE             | FP             | GO             | IOGA           | NP             | OA             |
| ERR2144882: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | ERROR          | ERROR          |
| ERR268390:  | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | SUCCESS        | SUCCESS        |
| ERR268421:  | INCOMPLETE_low | PARTIAL        | INCOMPLETE_low | INCOMPLETE_low | PARTIAL        | SUCCESS        |
| ERR845262:  | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        |
| ERR964904:  | ERROR          | INCOMPLETE_low | INCOMPLETE_low | ERROR          | PARTIAL        | ERROR          |
| SRR072296:  | PARTIAL        | PARTIAL        | INCOMPLETE_low | PARTIAL        | SUCCESS        | ERROR          |
| SRR1145775: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR1919543: | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        |
| SRR1925329: | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | PARTIAL        | PARTIAL        | PARTIAL        |
| SRR1946759: | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        | NO_PAIR        |
| SRR3089810: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | PARTIAL        | SUCCESS        |
| SRR3656962: | SUCCESS        | SUCCESS        | ERROR          | PARTIAL        | PARTIAL        | INCOMPLETE_low |
| SRR4099903: | PARTIAL        | SUCCESS        | PARTIAL        | ERROR          | INCOMPLETE_low | ERROR          |
| SRR4099904: | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | INCOMPLETE_low | ERROR          |
| SRR4428742: | PARTIAL        | SUCCESS        | PARTIAL        | ERROR          | SUCCESS        | PARTIAL        |
| SRR5028798: | SUCCESS        | SUCCESS        | PARTIAL        | PARTIAL        | SUCCESS        | PARTIAL        |
| SRR5152976: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602572: | SUCCESS        | PARTIAL        | ERROR          | ERROR          | PARTIAL        | ERROR          |
| SRR5602573: | PARTIAL        | PARTIAL        | ERROR          | ERROR          | INCOMPLETE_low | ERROR          |
| SRR5602574: | ERROR          | SUCCESS        | ERROR          | ERROR          | ERROR          | ERROR          |
| SRR5602575: | INCOMPLETE_low | SUCCESS        | ERROR          | ERROR          | ERROR          | ERROR          |
| SRR5602576: | ERROR          | PARTIAL        | SUCCESS        | ERROR          | ERROR          | INCOMPLETE_low |
| SRR5602577: | PARTIAL        | PARTIAL        | ERROR          | ERROR          | INCOMPLETE_low | ERROR          |
| SRR5602578: | INCOMPLETE_low | SUCCESS        | PARTIAL        | ERROR          | ERROR          | ERROR          |
| SRR5602579: | ERROR          | PARTIAL        | ERROR          | ERROR          | ERROR          | ERROR          |
| SRR5602581: | INCOMPLETE_low | SUCCESS        | PARTIAL        | ERROR          | ERROR          | ERROR          |
| SRR5602582: | SUCCESS        | SUCCESS        | PARTIAL        | PARTIAL        | ERROR          | ERROR          |
| SRR5602583: | ERROR          | INCOMPLETE_low | PARTIAL        | INCOMPLETE_low | ERROR          | ERROR          |
| SRR5602584: | SUCCESS        | SUCCESS        | PARTIAL        | ERROR          | INCOMPLETE_low | SUCCESS        |
| SRR5602585: | SUCCESS        | PARTIAL        | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602586: | INCOMPLETE_low | PARTIAL        | SUCCESS        | ERROR          | INCOMPLETE_low | PARTIAL        |
| SRR5602587: | ERROR          | PARTIAL        | PARTIAL        | ERROR          | ERROR          | ERROR          |
| SRR5602588: | SUCCESS        | PARTIAL        | ERROR          | PARTIAL        | INCOMPLETE_low | SUCCESS        |
| SRR5602589: | SUCCESS        | SUCCESS        | ERROR          | ERROR          | INCOMPLETE_low | SUCCESS        |
| SRR5602590: | PARTIAL        | PARTIAL        | ERROR          | PARTIAL        | ERROR          | ERROR          |
| SRR5602591: | INCOMPLETE_low | SUCCESS        | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602592: | SUCCESS        | SUCCESS        | ERROR          | PARTIAL        | PARTIAL        | ERROR          |
| SRR5602593: | SUCCESS        | SUCCESS        | PARTIAL        | PARTIAL        | ERROR          | ERROR          |
| SRR5602594: | INCOMPLETE_low | SUCCESS        | ERROR          | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602595: | ERROR          | INCOMPLETE_low | PARTIAL        | ERROR          | INCOMPLETE_low | ERROR          |
| SRR5602596: | INCOMPLETE_low | PARTIAL        | ERROR          | PARTIAL        | ERROR          | ERROR          |
| SRR5602597: | PARTIAL        | SUCCESS        | ERROR          | ERROR          | INCOMPLETE_low | ERROR          |
| SRR5602598: | ERROR          | SUCCESS        | ERROR          | PARTIAL        | ERROR          | INCOMPLETE_low |
| SRR5602599: | SUCCESS        | SUCCESS        | ERROR          | PARTIAL        | SUCCESS        | SUCCESS        |
| SRR5602600: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | ERROR          | ERROR          |
| SRR5602601: | PARTIAL        | PARTIAL        | PARTIAL        | ERROR          | ERROR          | ERROR          |
| SRR5602602: | SUCCESS        | SUCCESS        | ERROR          | PARTIAL        | SUCCESS        | SUCCESS        |
| SRR5602603: | ERROR          | SUCCESS        | ERROR          | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602604: | PARTIAL        | SUCCESS        | ERROR          | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602605: | SUCCESS        | PARTIAL        | PARTIAL        | ERROR          | PARTIAL        | ERROR          |
| SRR5602606: | INCOMPLETE_low | SUCCESS        | ERROR          | PARTIAL        | INCOMPLETE_low | SUCCESS        |
| SRR5602607: | INCOMPLETE_low | PARTIAL        | PARTIAL        | PARTIAL        | ERROR          | ERROR          |
| SRR5602608: | PARTIAL        | SUCCESS        | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          |
| SRR5602609: | ERROR          | SUCCESS        | ERROR          | PARTIAL        | ERROR          | ERROR          |
| SRR5602610: | ERROR          | PARTIAL        | PARTIAL        | INCOMPLETE_low | ERROR          | ERROR          |
| SRR954512:  | PARTIAL        | PARTIAL        | PARTIAL        | ERROR          | SUCCESS        | SUCCESS        |
| SRR955759:  | SUCCESS        | SUCCESS        | PARTIAL        | ERROR          | INCOMPLETE_low | SUCCESS        |



| Tool    | SUCCESS | %    | ERROR | PARTIAL | INCOMPl | NO_PAIR | Running | Total |
| CE      |      14 | ~26% |    11 |      17 |      12 |       3 |       0 |       |
| FP      |      31 | ~57% |     0 |      18 |       5 |       3 |       0 |       |
| GO      |       2 | ~4%  |    21 |      26 |       5 |       3 |       0 |       |
| IOGA    |       0 | ~0%  |    22 |      28 |       4 |       3 |       0 |       |
| NP      |       7 | ~13% |    19 |       8 |      20 |       3 |       0 |       |
| OA      |      11 | ~20% |    36 |       4 |       3 |       3 |       0 |       |
| Summary |      40 | ~74% |     - |       - |       - |       3 |       - |    57 | 
SUCCESS: one circular chloroplast in range
PARTIAL: more than one contig or non circular chloroplast in range
INCOMPLETE_HIGH / _LOW: more then one contig not in range
ERROR: no output / no chloroplasts found

After this first test runs, I used data from the GetOrganelle puplication (pre-print *CITATION*) to first further test the programs, and also to be able to reproduce the claimed success for the GetOrganelle.
Like in the privious tests the fast-plast and the chloroExtractor had the highest success rate. Also this two had the lowest error rate. The GetOrganell puplication claimed that these are all SRAs which 
the GetOrganelle would succed on, I was not able to reproduce this claim. But also the GetOrganelle is only tool which had an non automized step. They had to use Bandage to get their finished chloroplast.
This was done per hand on every on this SRAs and on 900 more. I looked into a few finished runs from GetOrganelle with Bandage. There were some which could be easily done per hand but also some which,
in my opinion could not be resolved per hand. So they may be tuned the GetOrganelle with different options for every single SRA, which is not the automized way I wanted to use. So I decided not to
use the GetOrganelle on further data. I also decided not to use IOGA, NOVOPlasty and ORG.Asm on the further steps on this research. IOGA tend to need a large amount of time compared to 
the other programs, for example the two simulated dataset (1:100, 1:1000) i canceled the runs after two whole weeks, also the the other simulated set could not be finished properly. It had the 
lowest success rate overall on both, the 1001 Genom Data and the GetOrganell Preprint data. NOVOPlasty also like Org.ASM had a lower successrate and a higher error rate then chloroExtractor.
  
 

*** Handling / installation 
**** Docker container -> Docker in Intro (?)
**** ( ref needed? -> in Tool introduction )
*** Time / run time
**** Time evaluation with `time` and $(date -Iseconds) 
*** resources needed (?)
**** Benchmark tests...?
still no data...
just estimated RAM values

We testet the several programs in successrate, RAM usage and time on our maschines. 
RAM is estimated -> 2M reads per file (2 files paired end Illumina ~200bp per read)
-FP ~60G RAM
-CE ~20G RAM
-GO ~50G
-OG ~30G
-NP ~30G
time can variate on which maschine it was running, CPU time? Time command



*** success rate
**** SUCCESS
one circular chloroplast in range
**** PARTIAL
more than one contig or non circular chloroplast in range
**** INCOMPLETE_HIGH / _LOW
more then one contig not in range
**** ERROR
no output / no chloroplasts found
**** AliTV pictures
*** standart parameters? 
**** how good does these work?
**** how easy to change?
*** automatisation -> High throughput
*** reproduceability
** different input Data / different sizes | amounts of Data
*** (Gold set ? )
*** sim reads
*** 1001 Genom paper
*** New ones (nocpbase)
*** GO preprint ( review...? / reproduceability...)
** Variance Calling
*** SNPs calling
*** Differences in the program runs? reproducable? assambler differences? heteroplasty/variants?
*** GWAS
** Structure Variance
* Results / Discussion 
** fast-plast and chloroExtractor are the best one, cuz success rate
*** AliTV pictures
** reproducable of fast-plast meh..., randomnes?
** 303 of 1135 (1001 Genom) chloroplasts
** 17 new chloroplasts (nocpbase)
** GO preprint.... meh GO is useless...
** SNP calling
** GWAS
** SV 


